<!DOCTYPE html>
<html lang="en-GB">
    <head>
        <title>
            Publications
        </title>
        <meta charset="UTF-8">
        <link rel="stylesheet" href="../css/style_main.css">
        <link rel="stylesheet" href="../css/colour_scheme.css">
        <script src="../timestamp.js"></script>
        <script src="../iframe.js" defer></script>
        <script>
        var name = "DataSHIELD";
        </script>
        <script src="search.js"></script>
        <style>
        button { height:2em; font-size:1em; margin-left:5px; } input#search { width:50%; }
        </style>
    </head>
    <body>
        <div id='header-container'>
            <div class='header width-master' role='banner'>
                <div id="main-logo">
                    <a accesskey="1" title="DataSHIELD" href="https://github.com/datashield"><img id="main-logo-img" src="../datashield_logo.svg" alt="Institution logo"></a>
                </div>
                <div class='maintitle' id='maintitle1'>
                    <span id='title1'><a href='../index.html'>DataSHIELD - Publications</a></span>
                </div>
            </div>
        </div>
        <div id="wrapper" class="width-master">
            <div id="col1" role="navigation">
                <!-- navigation object : navigation -->
                <ul class="navgroup">
                    <li>
                        <a href="../index.html">Home</a>
                    </li>
                    <li>
                        <a href="../about/index.html">About</a>
                    </li>
                    <li>
                        <a href="../about/project.html">About project</a>
                    </li>
                    <li>
                        <a href="../search/index.html">Search</a>
                    </li>
                    <li>
                        <a href="../tags/index.html">Tags</a>
                    </li>
                    <li>
                        <a href="../keywords/index.html">All Keywords</a>
                    </li>
                    <li>
                        <a href="../mesh/index.html">Major Keywords (MeSH)</a>
                    </li>
                    <li>
                        <a href="../country/index.html">Map by Country</a>
                    </li>
                    <li>
                        <a href="../metrics/index.html">Metrics</a>
                    </li>
                    <li>
                        <a href="../keyword_wordcloud/index.html">Keyword Cloud</a>
                    </li>
                    <li>
                        <a href="../abstractwordcloud/index.html">Abstract Word Cloud</a>
                    </li>
                </ul>
                <div class="after-navgroup">
                    <!-- navigation object : navigation bottom --><!-- start navigation : additional logo -->
                </div>
            </div>
            <p id="breadcrumbs">
                <a href="../index.html">Home</a> &gt; Search
            </p>
            <div id="content" role="main">
                <h1 id="pagetitle">
                    Search
                </h1>
                <p>
                    Search for the fields titles, abstracts, keywords and authors.<br>
                    To narrow down the results search for multiple fields at once.
                </p>
                <p>
                    <input type="text" id="search"><button onclick="search();">Search</button>
                </p>
                <div style="display:none;" id="searching">
                    Searching...
                </div>
                <h2 id="num_search_results"></h2>
                <div id="search_results"></div>
                <script id="search_data">
                var papers = [{"IDs": {"DOI": "10.1007/s10654-024-01126-4", "PMID": "38805076"}, "title": "Stress and anxiety during pregnancy and length of gestation: a federated study using data from five Canadian and European birth cohorts.", "abstract": "['While its etiology is not fully elucidated, preterm birth represents a major public health concern as it is the leading cause of child mortality and morbidity. Stress is one of the most common perinatal conditions and may increase the risk of preterm birth. In this paper we aimed to investigate the association of maternal perceived stress and anxiety with length of gestation. We used harmonized data from five birth cohorts from Canada, France, and Norway. A total of 5297 pregnancies of singletons were included in the analysis of perceived stress and gestational duration, and 55,775 pregnancies for anxiety. Federated analyses were performed through the DataSHIELD platform using Cox regression models within intervals of gestational age. The models were fit for each cohort separately, and the cohort-specific results were combined using random effects study-level meta-analysis. Moderate and high levels of perceived stress during pregnancy were associated with a shorter length of gestation in the very/moderately preterm interval [moderate: hazard ratio (HR) 1.92 (95%CI 0.83, 4.48); high: 2.04 (95%CI 0.77, 5.37)], albeit not statistically significant. No association was found for the other intervals. Anxiety was associated with gestational duration in the very/moderately preterm interval [1.66 (95%CI 1.32, 2.08)], and in the early term interval [1.15 (95%CI 1.08, 1.23)]. Our findings suggest that perceived stress and anxiety are associated with an increased risk of earlier birth, but only in the earliest gestational ages. We also found an association in the early term period for anxiety, but the result was only driven by the largest cohort, which collected information the latest in pregnancy. This raised a potential issue of reverse causality as anxiety later in pregnancy could be due to concerns about early signs of a possible preterm birth.']", "keywords": {"mesh": [], "other": ["Anxiety", "Data harmonization", "Gestational age at birth", "Multi cohort study", "Perceived stress"]}, "full_author_list": ["Bergeron J", "Avraam D", "Calas L", "Fraser W", "Harris J", "Heude B", "Mandhane P", "Moraes T", "Muckle G", "Nader J", "S\u00e9guin J", "Simons E", "Subbarao P", "Swertz M", "Tough S", "Turvey S", "Fortier I", "Rod N", "Andersen A"], "journal": {"journal_name": "Eur J Epidemiol", "volume": "", "issue": ""}}, {"IDs": {"DOI": "10.1080/03009734.2018.1556754", "PMID": "30618330"}, "title": "Rationale for a Swedish cohort consortium.", "abstract": "[\"We herein outline the rationale for a Swedish cohort consortium, aiming to facilitate greater use of Swedish cohorts for world-class research. Coordination of all Swedish prospective population-based cohorts in a common infrastructure would enable more precise research findings and facilitate research on rare exposures and outcomes, leading to better utilization of study participants' data, better return of funders' investments, and higher benefit to patients and populations. We motivate the proposed infrastructure partly by lessons learned from a pilot study encompassing data from 21 cohorts. We envisage a standing Swedish cohort consortium that would drive development of epidemiological research methods and strengthen the Swedish as well as international epidemiological competence, community, and competitiveness.\"]", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "N"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Ethics, Medical", "major": "N"}, {"term": "Genetic Association Studies", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "International Cooperation", "major": "N"}, {"term": "Outcome Assessment, Health Care", "major": "N"}, {"term": "Pilot Projects", "major": "N"}, {"term": "Proportional Hazards Models", "major": "N"}, {"term": "Prospective Studies", "major": "N"}, {"term": "Registries", "major": "Y"}, {"term": "Research Design", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Subarachnoid Hemorrhage", "major": "N"}, {"term": "Sweden", "major": "N"}], "other": ["Common infrastructure", "Swedish cohort consortium", "epidemiological research", "pilot study", "rare outcomes"]}, "full_author_list": ["Sundstr\u00f6m J", "Bj\u00f6rkelund C", "Giedraitis V", "Hansson P", "H\u00f6gman M", "Janson C", "Koupil I", "Kristenson M", "Trolle Lagerros Y", "Leppert J", "Lind L", "Lissner L", "Johansson I", "Ludvigsson J", "Nilsson P", "Olsson H", "Pedersen N", "Rosenblad A", "Rosengren A", "Sandin S", "Sn\u00e4ckerstr\u00f6m T", "Stenbeck M", "S\u00f6derberg S", "Weiderpass E", "Wanhainen A", "Wennberg P", "Fortier I", "Heller S", "Storg\u00e4rds M", "Svennblad B"], "journal": {"journal_name": "Ups J Med Sci", "volume": "124", "issue": "1"}}, {"IDs": {"DOI": "10.1093/ije/dyx180", "PMID": "29025122"}, "title": "Software Application Profile: Opal and Mica: open-source software solutions for epidemiological data management, harmonization and dissemination.", "abstract": "['Improving the dissemination of information on existing epidemiological studies and facilitating the interoperability of study databases are essential to maximizing the use of resources and accelerating improvements in health. To address this, Maelstrom Research proposes Opal and Mica, two inter-operable open-source software packages providing out-of-the-box solutions for epidemiological data management, harmonization and dissemination.', 'Opal and Mica are two standalone but inter-operable web applications written in Java, JavaScript and PHP. They provide web services and modern user interfaces to access them.', 'Opal allows users to import, manage, annotate and harmonize study data. Mica is used to build searchable web portals disseminating study and variable metadata. When used conjointly, Mica users can securely query and retrieve summary statistics on geographically dispersed Opal servers in real-time. Integration with the DataSHIELD approach allows conducting more complex federated analyses involving statistical models.', 'Opal and Mica are open-source and freely available at [www.obiba.org] under a General Public License (GPL) version 3, and the metadata models and taxonomies that accompany them are available under a Creative Commons licence.']", "keywords": {"mesh": [{"term": "Canada", "major": "N"}, {"term": "Database Management Systems", "major": "Y"}, {"term": "Epidemiologic Studies", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Dissemination", "major": "N"}, {"term": "Internet", "major": "N"}, {"term": "Software", "major": "Y"}], "other": []}, "full_author_list": ["Doiron D", "Marcon Y", "Fortier I", "Burton P", "Ferretti V"], "journal": {"journal_name": "Int J Epidemiol", "volume": "46", "issue": "5"}}, {"IDs": {"DOI": "10.2188/jea.JE20210241", "PMID": "34776498"}, "title": "Measures of Early-life Behavior and Later Psychopathology in the LifeCycle Project - EU Child Cohort Network: A Cohort Description.", "abstract": "['The EU LifeCycle Project was launched in 2017 to combine, harmonize, and analyze data from more than 250,000 participants across Europe and Australia, involving cohorts participating in the EU-funded LifeCycle Project. The purpose of this cohort description is to provide a detailed overview of the major measures within mental health domains that are available in 17 European and Australian cohorts participating in the LifeCycle Project.', 'Data on cognitive, behavioral, and psychological development has been collected on participants from birth until adulthood through questionnaire and medical data. We developed an inventory of the available data by mapping individual instruments, domain types, and age groups, providing the basis for statistical harmonization across mental health measures.', 'The mental health data in LifeCycle contain longitudinal and cross-sectional data from birth throughout the life course, covering domains across a wide range of behavioral and psychopathology indicators and outcomes, including executive function, depression, ADHD, and cognition. These data span a unique combination of qualitative data collected through behavioral/cognitive/mental health questionnaires and examination, as well as data from biological samples and indices in the form of imaging (MRI, fetal ultrasound) and DNA methylation data. Harmonized variables on a subset of mental health domains have been developed, providing statistical equivalence of measures required for longitudinal meta-analyses across instruments and cohorts.', 'Mental health data harmonized through the LifeCycle project can be used to study life-course trajectories and exposure-outcome models that examine early life risk factors for mental illness and develop predictive markers for later-life disease.']", "keywords": {"mesh": [{"term": "Humans", "major": "N"}, {"term": "Child", "major": "N"}, {"term": "Adult", "major": "N"}, {"term": "Cross-Sectional Studies", "major": "N"}, {"term": "Australia", "major": "N"}, {"term": "Japan", "major": "N"}, {"term": "Mental Disorders", "major": "Y"}, {"term": "Mental Health", "major": "N"}], "other": ["DataSHIELD", "birth and pregnancy cohorts", "child behavior and mental health", "child development", "population epidemiology"]}, "full_author_list": ["Nader J", "L\u00f3pez-Vicente M", "Julvez J", "Guxens M", "Cadman T", "Elhakeem A", "J\u00e4rvelin M", "Rautio N", "Miettunen J", "El Marroun H", "Melchior M", "Heude B", "Charles M", "Yang T", "McEachan R", "Wright J", "Polanska K", "Carson J", "Lin A", "Rauschert S", "Huang R", "Popovic M", "Richiardi L", "Corpeleijn E", "Cardol M", "Mikkola T", "Eriksson J", "Salika T", "Inskip H", "Vinther J", "Strandberg-Larsen K", "G\u00fcrlich K", "Grote V", "Koletzko B", "Vafeiadi M", "Sunyer J", "Jaddoe V", "Harris J"], "journal": {"journal_name": "J Epidemiol", "volume": "33", "issue": "6"}}, {"IDs": {"DOI": "10.1038/s41467-023-38148-7", "PMID": "37137910"}, "title": "Identification of biomarkers for glycaemic deterioration in type 2 diabetes.", "abstract": "['We identify biomarkers for disease progression in three type 2 diabetes cohorts encompassing 2,973 individuals across three molecular classes, metabolites, lipids and proteins. Homocitrulline, isoleucine and 2-aminoadipic acid, eight triacylglycerol species, and lowered sphingomyelin 42:2;2 levels are predictive of faster progression towards insulin requirement. Of ~1,300 proteins examined in two cohorts, levels of GDF15/MIC-1, IL-18Ra, CRELD1, NogoR, FAS, and ENPP7 are associated with faster progression, whilst SMAC/DIABLO, SPOCK1 and HEMK2 predict lower progression rates. In an external replication, proteins and lipids are associated with diabetes incidence and prevalence. NogoR/RTN4R injection improved glucose tolerance in high fat-fed male mice but impaired it in male db/db mice. High NogoR levels led to islet cell apoptosis, and IL-18R antagonised inflammatory IL-18 signalling towards nuclear factor kappa-B in vitro. This comprehensive, multi-disciplinary approach thus identifies biomarkers with potential prognostic utility, provides evidence for possible disease mechanisms, and identifies potential therapeutic avenues to slow diabetes progression.']", "keywords": {"mesh": [{"term": "Mice", "major": "N"}, {"term": "Animals", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Diabetes Mellitus, Type 2", "major": "Y"}, {"term": "Blood Glucose", "major": "N"}, {"term": "Islets of Langerhans", "major": "Y"}, {"term": "Insulin", "major": "N"}, {"term": "Lipids", "major": "N"}, {"term": "Biomarkers", "major": "N"}, {"term": "Cell Adhesion Molecules", "major": "N"}, {"term": "Extracellular Matrix Proteins", "major": "N"}], "other": []}, "full_author_list": ["Slieker R", "Donnelly L", "Akalestou E", "Lopez-Noriega L", "Melhem R", "G\u00fcne\u015f A", "Abou Azar F", "Efanov A", "Georgiadou E", "Muniangi-Muhitu H", "Sheikh M", "Giordano G", "\u00c5kerlund M", "Ahlqvist E", "Ali A", "Banasik K", "Brunak S", "Barovic M", "Bouland G", "Burdet F", "Canouil M", "Dragan I", "Elders P", "Fernandez C", "Festa A", "Fitipaldi H", "Froguel P", "Gudmundsdottir V", "Gudnason V", "Gerl M", "van der Heijden A", "Jennings L", "Hansen M", "Kim M", "Leclerc I", "Klose C", "Kuznetsov D", "Mansour Aly D", "Mehl F", "Marek D", "Melander O", "Niknejad A", "Ottosson F", "Pavo I", "Duffin K", "Syed S", "Shaw J", "Cabrera O", "Pullen T", "Simons K", "Solimena M", "Suvitaival T", "Wretlind A", "Rossing P", "Lyssenko V", "Legido Quigley C", "Groop L", "Thorens B", "Franks P", "Lim G", "Estall J", "Ibberson M", "Beulens J", "'t Hart L", "Pearson E", "Rutter G"], "journal": {"journal_name": "Nat Commun", "volume": "14", "issue": "1"}}, {"IDs": {"DOI": "10.1093/ije/dyu188", "PMID": "25261970"}, "title": "DataSHIELD: taking the analysis to the data, not the data to the analysis.", "abstract": "[\"Research in modern biomedicine and social science requires sample sizes so large that they can often only be achieved through a pooled co-analysis of data from several studies. But the pooling of information from individuals in a central database that may be queried by researchers raises important ethico-legal questions and can be controversial. In the UK this has been highlighted by recent debate and controversy relating to the UK's proposed 'care.data' initiative, and these issues reflect important societal and professional concerns about privacy, confidentiality and intellectual property. DataSHIELD provides a novel technological solution that can circumvent some of the most basic challenges in facilitating the access of researchers and other healthcare professionals to individual-level data.\", 'Commands are sent from a central analysis computer (AC) to several data computers (DCs) storing the data to be co-analysed. The data sets are analysed simultaneously but in parallel. The separate parallelized analyses are linked by non-disclosive summary statistics and commands transmitted back and forth between the DCs and the AC. This paper describes the technical implementation of DataSHIELD using a modified R statistical environment linked to an Opal database deployed behind the computer firewall of each DC. Analysis is controlled through a standard R environment at the AC.', 'Based on this Opal/R implementation, DataSHIELD is currently used by the Healthy Obese Project and the Environmental Core Project (BioSHaRE-EU) for the federated analysis of 10 data sets across eight European countries, and this illustrates the opportunities and challenges presented by the DataSHIELD approach.', 'DataSHIELD facilitates important research in settings where: (i) a co-analysis of individual-level data from several studies is scientifically necessary but governance restrictions prohibit the release or sharing of some of the required data, and/or render data access unacceptably slow; (ii) a research group (e.g. in a developing nation) is particularly vulnerable to loss of intellectual property-the researchers want to fully share the information held in their data with national and international collaborators, but do not wish to hand over the physical data themselves; and (iii) a data set is to be included in an individual-level co-analysis but the physical size of the data precludes direct transfer to a new site for analysis.']", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "Y"}, {"term": "Computational Biology", "major": "N"}, {"term": "Computer Security", "major": "Y"}, {"term": "Confidentiality", "major": "Y"}, {"term": "Databases, Factual", "major": "N"}, {"term": "Datasets as Topic", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Information Storage and Retrieval", "major": "Y"}, {"term": "United Kingdom", "major": "N"}], "other": ["DataSHIELD", "ELSI", "bioinformatics", "confidentiality", "disclosure", "distributed computing", "intellectual property", "pooled analysis", "privacy"]}, "full_author_list": ["Gaye A", "Marcon Y", "Isaeva J", "LaFlamme P", "Turner A", "Jones E", "Minion J", "Boyd A", "Newby C", "Nuotio M", "Wilson R", "Butters O", "Murtagh B", "Demir I", "Doiron D", "Giepmans L", "Wallace S", "Budin-Lj\u00f8sne I", "Oliver Schmidt C", "Boffetta P", "Boniol M", "Bota M", "Carter K", "deKlerk N", "Dibben C", "Francis R", "Hiekkalinna T", "Hveem K", "Kval\u00f8y K", "Millar S", "Perry I", "Peters A", "Phillips C", "Popham F", "Raab G", "Reischl E", "Sheehan N", "Waldenberger M", "Perola M", "van den Heuvel E", "Macleod J", "Knoppers B", "Stolk R", "Fortier I", "Harris J", "Woffenbuttel B", "Murtagh M", "Ferretti V", "Burton P"], "journal": {"journal_name": "Int J Epidemiol", "volume": "43", "issue": "6"}}, {"IDs": {"DOI": "10.1186/s12911-022-01754-4", "PMID": "35090447"}, "title": "A deterministic approach for protecting privacy in sensitive personal data.", "abstract": "['Data privacy is one of the biggest challenges for any organisation which processes personal data, especially in the area of medical research where data include sensitive information about patients and study participants. Sharing of data is therefore problematic, which is at odds with the principle of open data that is so important to the advancement of society and science. Several statistical methods and computational tools have been developed to help data custodians and analysts overcome this challenge.', 'In this paper, we propose a new deterministic approach for anonymising personal data. The method stratifies the underlying data by the categorical variables and re-distributes the continuous variables through a k nearest neighbours based algorithm.', 'We demonstrate the use of the deterministic anonymisation on real data, including data from a sample of Titanic passengers, and data from participants in the 1958 Birth Cohort.', 'The proposed procedure makes data re-identification difficult while minimising the loss of utility (by preserving the spatial properties of the underlying data); the latter means that informative statistical analysis can still be conducted.']", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "Y"}, {"term": "Data Anonymization", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Privacy", "major": "Y"}], "other": ["Data privacy", "Deterministic anonymisation", "Disclosure risk", "Information loss", "k nearest neighbours"]}, "full_author_list": ["Avraam D", "Jones E", "Burton P"], "journal": {"journal_name": "BMC Med Inform Decis Mak", "volume": "22", "issue": "1"}}, {"IDs": {"DOI": "10.1111/bpa.13228", "PMID": "38012085"}, "title": "DistSNE: Distributed computing and online visualization of DNA methylation-based central nervous system tumor classification.", "abstract": "['The current state-of-the-art analysis of central nervous system (CNS) tumors through DNA methylation profiling relies on the tumor classifier developed by Capper and colleagues, which centrally harnesses DNA methylation data provided by users. Here, we present a distributed-computing-based approach for CNS tumor classification that achieves a comparable performance to centralized systems while safeguarding privacy. We utilize the t-distributed neighborhood embedding (t-SNE) model for dimensionality reduction and visualization of tumor classification results in two-dimensional graphs in a distributed approach across multiple sites (DistSNE). DistSNE provides an intuitive web interface (https://gin-tsne.med.uni-giessen.de) for user-friendly local data management and federated methylome-based tumor classification calculations for multiple collaborators in a DataSHIELD environment. The freely accessible web interface supports convenient data upload, result review, and summary report generation. Importantly, increasing sample size as achieved through distributed access to additional datasets allows DistSNE to improve cluster analysis and enhance predictive power. Collectively, DistSNE enables a simple and fast classification of CNS tumors using large-scale methylation data from distributed sources, while maintaining the privacy and allowing easy and flexible network expansion to other institutes. This approach holds great potential for advancing human brain tumor classification and fostering collaborative precision medicine in neuro-oncology.']", "keywords": {"mesh": [{"term": "Humans", "major": "N"}, {"term": "DNA Methylation", "major": "N"}, {"term": "Central Nervous System Neoplasms", "major": "Y"}, {"term": "Brain Neoplasms", "major": "Y"}], "other": ["DataSHIELD", "brain tumor classification", "distributed computing", "methylome", "tSNE", "webinterface"]}, "full_author_list": ["Schmid K", "Sehring J", "N\u00e9meth A", "Harter P", "Weber K", "Vengadeswaran A", "Storf H", "Seidemann C", "Karki K", "Fischer P", "Dohmen H", "Selignow C", "von Deimling A", "Grau S", "Schr\u00f6der U", "Plate K", "Stein M", "Uhl E", "Acker T", "Amsel D"], "journal": {"journal_name": "Brain Pathol", "volume": "34", "issue": "3"}}, {"IDs": {"DOI": "10.1371/journal.pmed.1004036", "PMID": "36701266"}, "title": "Gestational age at birth and body size from infancy through adolescence: An individual participant data meta-analysis on 253,810 singletons in 16 birth cohort studies.", "abstract": "['Preterm birth is the leading cause of perinatal morbidity and mortality and is associated with adverse developmental and long-term health outcomes, including several cardiometabolic risk factors and outcomes. However, evidence about the association of preterm birth with later body size derives mainly from studies using birth weight as a proxy of prematurity rather than an actual length of gestation. We investigated the association of gestational age (GA) at birth with body size from infancy through adolescence.', \"We conducted a two-stage individual participant data (IPD) meta-analysis using data from 253,810 mother-child dyads from 16 general population-based cohort studies in Europe (Denmark, Finland, France, Italy, Norway, Portugal, Spain, the Netherlands, United Kingdom), North America (Canada), and Australasia (Australia) to estimate the association of GA with body mass index (BMI) and overweight (including obesity) adjusted for the following maternal characteristics as potential confounders: education, height, prepregnancy BMI, ethnic background, parity, smoking during pregnancy, age at child's birth, gestational diabetes and hypertension, and preeclampsia. Pregnancy and birth cohort studies from the LifeCycle and the EUCAN-Connect projects were invited and were eligible for inclusion if they had information on GA and minimum one measurement of BMI between infancy and adolescence. Using a federated analytical tool (DataSHIELD), we fitted linear and logistic regression models in each cohort separately with a complete-case approach and combined the regression estimates and standard errors through random-effects study-level meta-analysis providing an overall effect estimate at early infancy (&gt;0.0 to 0.5 years), late infancy (&gt;0.5 to 2.0 years), early childhood (&gt;2.0 to 5.0 years), mid-childhood (&gt;5.0 to 9.0 years), late childhood (&gt;9.0 to 14.0 years), and adolescence (&gt;14.0 to 19.0 years). GA was positively associated with BMI in the first decade of life, with the greatest increase in mean BMI z-score during early infancy (0.02, 95% confidence interval (CI): 0.00; 0.05, p &lt; 0.05) per week of increase in GA, while in adolescence, preterm individuals reached similar levels of BMI (0.00, 95% CI: -0.01; 0.01, p 0.9) as term counterparts. The association between GA and overweight revealed a similar pattern of association with an increase in odds ratio (OR) of overweight from late infancy through mid-childhood (OR 1.01 to 1.02) per week increase in GA. By adolescence, however, GA was slightly negatively associated with the risk of overweight (OR 0.98 [95% CI: 0.97; 1.00], p 0.1) per week of increase in GA. Although based on only four cohorts (n = 32,089) that reached the age of adolescence, data suggest that individuals born very preterm may be at increased odds of overweight (OR 1.46 [95% CI: 1.03; 2.08], p &lt; 0.05) compared with term counterparts. Findings were consistent across cohorts and sensitivity analyses despite considerable heterogeneity in cohort characteristics. However, residual confounding may be a limitation in this study, while findings may be less generalisable to settings in low- and middle-income countries.\", 'This study based on data from infancy through adolescence from 16 cohort studies found that GA may be important for body size in infancy, but the strength of association attenuates consistently with age. By adolescence, preterm individuals have on average a similar mean BMI to peers born at term.']", "keywords": {"mesh": [{"term": "Child", "major": "N"}, {"term": "Pregnancy", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Infant, Newborn", "major": "N"}, {"term": "Infant", "major": "N"}, {"term": "Child, Preschool", "major": "N"}, {"term": "Adolescent", "major": "N"}, {"term": "Overweight", "major": "Y"}, {"term": "Gestational Age", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Premature Birth", "major": "Y"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Birth Weight", "major": "N"}, {"term": "Body Mass Index", "major": "N"}], "other": []}, "full_author_list": ["Vinther J", "Cadman T", "Avraam D", "Ekstr\u00f8m C", "S\u00f8rensen T", "Elhakeem A", "Santos A", "Pinot de Moira A", "Heude B", "I\u00f1iguez C", "Pizzi C", "Simons E", "Voerman E", "Corpeleijn E", "Zariouh F", "Santorelli G", "Inskip H", "Barros H", "Carson J", "Harris J", "Nader J", "Ronkainen J", "Strandberg-Larsen K", "Santa-Marina L", "Calas L", "Cederkvist L", "Popovic M", "Charles M", "Welten M", "Vrijheid M", "Azad M", "Subbarao P", "Burton P", "Mandhane P", "Huang R", "Wilson R", "Haakma S", "Fern\u00e1ndez-Barr\u00e9s S", "Turvey S", "Santos S", "Tough S", "Sebert S", "Moraes T", "Salika T", "Jaddoe V", "Lawlor D", "Nybo Andersen A"], "journal": {"journal_name": "PLoS Med", "volume": "20", "issue": "1"}}, {"IDs": {"DOI": "10.1186/s13104-022-06111-2", "PMID": "35761417"}, "title": "dsSynthetic: synthetic data generation for the DataSHIELD federated analysis system.", "abstract": "['Platforms such as DataSHIELD allow users to analyse sensitive data remotely, without having full access to the detailed data items (federated analysis). While this feature helps to overcome difficulties with data sharing, it can make it challenging to write code without full visibility of the data. One solution is to generate realistic, non-disclosive synthetic data that can be transferred to the analyst so they can perfect their code without the access limitation. When this process is complete, they can run the code on the real data.', 'We have created a package in DataSHIELD (dsSynthetic) which allows generation of realistic synthetic data, building on existing packages. In our paper and accompanying tutorial we demonstrate how the use of synthetic data generated with our package can help DataSHIELD users with tasks such as writing analysis scripts and harmonising data to common scales and measures.']", "keywords": {"mesh": [{"term": "Information Dissemination", "major": "Y"}], "other": ["Data harmonisation", "Federated analysis", "Synthetic data"]}, "full_author_list": ["Banerjee S", "Bishop T"], "journal": {"journal_name": "BMC Res Notes", "volume": "15", "issue": "1"}}, {"IDs": {"DOI": "10.1093/ije/dyq111", "PMID": "20630989"}, "title": "DataSHIELD: resolving a conflict in contemporary bioscience--performing a pooled analysis of individual-level data without sharing the data.", "abstract": "['Contemporary bioscience sometimes demands vast sample sizes and there is often then no choice but to synthesize data across several studies and to undertake an appropriate pooled analysis. This same need is also faced in health-services and socio-economic research. When a pooled analysis is required, analytic efficiency and flexibility are often best served by combining the individual-level data from all sources and analysing them as a single large data set. But ethico-legal constraints, including the wording of consent forms and privacy legislation, often prohibit or discourage the sharing of individual-level data, particularly across national or other jurisdictional boundaries. This leads to a fundamental conflict in competing public goods: individual-level analysis is desirable from a scientific perspective, but is prevented by ethico-legal considerations that are entirely valid.', 'Data aggregation through anonymous summary-statistics from harmonized individual-level databases (DataSHIELD), provides a simple approach to analysing pooled data that circumvents this conflict. This is achieved via parallelized analysis and modern distributed computing and, in one key setting, takes advantage of the properties of the updating algorithm for generalized linear models (GLMs).', 'The conceptual use of DataSHIELD is illustrated in two different settings.', 'As the study of the aetiological architecture of chronic diseases advances to encompass more complex causal pathways-e.g. to include the joint effects of genes, lifestyle and environment-sample size requirements will increase further and the analysis of pooled individual-level data will become ever more important. An aim of this conceptual article is to encourage others to address the challenges and opportunities that DataSHIELD presents, and to explore potential extensions, for example to its use when different data sources hold different data on the same individuals.']", "keywords": {"mesh": [{"term": "Causality", "major": "N"}, {"term": "Confidentiality", "major": "N"}, {"term": "Epidemiologic Methods", "major": "Y"}, {"term": "Ethics, Research", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Storage and Retrieval", "major": "N"}, {"term": "Meta-Analysis as Topic", "major": "Y"}, {"term": "Research Design", "major": "N"}], "other": []}, "full_author_list": ["Wolfson M", "Wallace S", "Masca N", "Rowe G", "Sheehan N", "Ferretti V", "LaFlamme P", "Tobin M", "Macleod J", "Little J", "Fortier I", "Knoppers B", "Burton P"], "journal": {"journal_name": "Int J Epidemiol", "volume": "39", "issue": "5"}}, {"IDs": {"DOI": "10.14301/llcs.v9i4.478", "PMID": ""}, "title": "A software package for the application of probabilistic anonymisation to sensitive individual-level data: a proof of principle with an example from the ALSPAC birth cohort study", "abstract": "Individual-level data require protection from unauthorised access to safeguard confidentiality and security of sensitive information. Risks of disclosure are evaluated through privacy risk assessments and are controlled or minimised before data sharing and integration. The evolution from \u2018Micro Data Laboratory\u2019 traditions (i.e. access in controlled physical locations) to \u2018Open Data\u2019 (i.e. sharing individual-level data) drives the development of efficient anonymisation methods and protection controls. Effective anonymisation techniques should increase the uncertainty surrounding re-identification while retaining data utility, allowing informative data analysis. \u2018Probabilistic anonymisation\u2019 is one such technique, which alters the data by addition of random noise. In this paper, we describe the implementation of one probabilistic anonymisation technique into an operational software written in R and we demonstrate its applicability through application to analysis of asthma-related data from the ALSPAC cohort study. The software is designed to be used by data managers and users without the requirement of advanced statistical knowledge.", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Avraam D", "Boyd A", "Goldstein H", "Burton P"], "journal": {"journal_name": "Longitudinal and Life Course Studies", "volume": "9", "issue": "4"}}, {"IDs": {"DOI": "10.1016/j.jaci.2022.01.023", "PMID": "35150722"}, "title": "Associations of early-life pet ownership with asthma and allergic sensitization: A\u00a0meta-analysis of more than 77,000 children from the EU Child Cohort Network.", "abstract": "['Studies examining associations of early-life cat and dog ownership with childhood asthma have reported inconsistent results. Several factors could explain these inconsistencies, including type of pet, timing, and degree of exposure.', 'Our aim was to study associations of early-life cat and dog ownership with asthma in school-aged children, including the role of type (cat vs dog), timing (never, prenatal, or early childhood), and degree of ownership (number of pets owned), and the role of allergic sensitization.', 'We used harmonized data from 77,434 mother-child dyads from 9 birth cohorts in the European Union Child Cohort Network when the child was 5 to 11 years old. Associations were examined through the DataSHIELD platform by using adjusted logistic regression models, which were fitted separately for each cohort and combined by using random effects meta-analysis.', 'The prevalence of early-life cat and dog ownership ranged from 12% to 45% and 7% to 47%, respectively, and the prevalence of asthma ranged from 2% to 20%. There was no overall association between either cat or dog ownership and asthma (odds ratio [OR]\\xa0= 0.97 [95% CI\\xa0= 0.87-1.09] and 0.92 [95% CI\\xa0= 0.85-1.01], respectively). Timing and degree of ownership did not strongly influence associations. Cat and dog ownership were also not associated with cat- and dog-specific allergic sensitization (OR\\xa0= 0.92 [95% CI\\xa0= 0.75-1.13] and 0.93 [95% CI\\xa0= 0.57-1.54], respectively). However, cat- and dog-specific allergic sensitization was strongly associated with school-age asthma (OR\\xa0= 6.69 [95% CI\\xa0= 4.91-9.10] and 5.98 [95% CI\\xa0= 3.14-11.36], respectively). There was also some indication of an interaction between ownership and sensitization, suggesting that ownership may exacerbate the risks associated with pet-specific sensitization but offer some protection against asthma in the absence of sensitization.', 'Our findings do not support early-life cat and dog ownership in themselves increasing the risk of school-age asthma, but they do suggest that ownership may potentially\\xa0exacerbate the risks associated with cat- and dog-specific allergic sensitization.']", "keywords": {"mesh": [{"term": "Allergens", "major": "Y"}, {"term": "Animals", "major": "N"}, {"term": "Asthma", "major": "Y"}, {"term": "Cats", "major": "N"}, {"term": "Child", "major": "N"}, {"term": "Child, Preschool", "major": "N"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Dogs", "major": "N"}, {"term": "Environmental Exposure", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Odds Ratio", "major": "N"}, {"term": "Ownership", "major": "N"}], "other": ["Cat", "FAIR (findable", "accessible", "allergic sensitization", "and reusable)", "asthma", "birth cohort", "children", "dog", "exposure", "interoperable", "life course epidemiology", "meta-analysis", "ownership"]}, "full_author_list": ["Pinot de Moira A", "Strandberg-Larsen K", "Bishop T", "Pedersen M", "Avraam D", "Cadman T", "Calas L", "Casas M", "de Lauzon Guillain B", "Elhakeem A", "Esplugues A", "Estarlich M", "Foong R", "Haakma S", "Harris J", "Huang R", "Inskip H", "Lertxundi A", "Mensink-Bout S", "Nader J", "Pizzi C", "Popovic M", "Salika T", "Sunyer J", "Van Meel E", "Swertz M", "Jaddoe V", "Burton P", "Duijts L", "Nybo Andersen A"], "journal": {"journal_name": "J Allergy Clin Immunol", "volume": "150", "issue": "1"}}, {"IDs": {"DOI": "10.1093/jn/nxz194", "PMID": "31504715"}, "title": "Dietary Patterns Are Associated with Serum Metabolite Patterns and Their Association Is Influenced by Gut Bacteria among Older German Adults.", "abstract": "['Although dietary intakes and dietary intake patterns (DPs) have been associated with single metabolites, it is unclear whether DPs are also reflected in specific metabolite patterns (MPs). Moreover, the influence of groups of gut bacteria on the relationship between DPs and MPs is underexplored.', 'We aimed to investigate the association of DPs and serum MPs and also the modifying effect of the gut bacteria compositional patterns (BCPs).', 'This is a cross-sectional investigation among 225 individuals (median age: 63 y; 53% women) from the European Prospective Investigation into Cancer and Nutrition study. Dietary intakes were assessed by three 24-h dietary recalls, gut bacteria composition was quantified by 16S rRNA gene sequencing, and the serum metabolome was profiled by an untargeted approach. We identified DPs and BCPs by the treelet transform analysis. We modeled associations between DPs and 8 previously published MPs and the modifying effect of BCPs by fitting generalized linear models using DataSHIELD R.', 'We identified 5 DPs and 7 BCPs. The \"bread, margarine, and processed meat\" and \"fruiting vegetables and vegetable oils\" DPs were positively associated with the \"amino acids\" (\u03b2\\xa0=\\xa00.35; 95% CI: 0.02, 0.69; P\\xa0=\\xa00.03) and \"fatty acids\" MPs (\u03b2\\xa0=\\xa00.45; 95% CI: 0.16, 0.74; P\\xa0=\\xa00.01), respectively. The \"tea and miscellaneous\" was inversely associated with the \"amino acids\" (\u03b2\\xa0=\\xa0-0.28; 95% CI: -0.52, -0.05; P\\xa0=\\xa00.02) and \"amino acid derivatives\" MPs (\u03b2\\xa0=\\xa0-0.21; 95% CI: -0.39, -0.02; P\\xa0=\\xa00.03). One BCP negatively modified the association between the \"bread, margarine, and processed meat\" DP and the \"amino acids\" MP (P-interaction\\xa0=\\xa00.01).', 'In older German adults, DPs are reflected in MPs, and the gut bacteria attenuate 1 DP-MP association. These MPs should be explored as biomarkers of these jointly consumed foods while taking into account a potentially modifying role of the gut bacteria.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Aged", "major": "N"}, {"term": "Biomarkers", "major": "N"}, {"term": "Cross-Sectional Studies", "major": "N"}, {"term": "Diet", "major": "Y"}, {"term": "Feeding Behavior", "major": "Y"}, {"term": "Female", "major": "N"}, {"term": "Food", "major": "N"}, {"term": "Gastrointestinal Microbiome", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Middle Aged", "major": "N"}], "other": ["DataSHIELD", "dietary intake patterns", "gut bacteria compositional patterns", "serum metabolite patterns", "treelet transform analysis"]}, "full_author_list": ["Oluwagbemigun K", "Foerster J", "Watkins C", "Fouhy F", "Stanton C", "Bergmann M", "Boeing H", "N\u00f6thlings U"], "journal": {"journal_name": "J Nutr", "volume": "150", "issue": "1"}}, {"IDs": {"DOI": "10.3390/nu13041223", "PMID": "33917229"}, "title": "Heterogeneity of Associations between Total and Types of Fish Intake and the Incidence of Type 2 Diabetes: Federated Meta-Analysis of 28 Prospective Studies Including 956,122 Participants.", "abstract": "['The association between fish consumption and new-onset type 2 diabetes is inconsistent and differs according to geographical location. We examined the association between the total and types of fish consumption and type 2 diabetes using individual participant data from 28 prospective cohort studies from the Americas (6), Europe (15), the Western Pacific (6), and the Eastern Mediterranean (1) comprising 956,122 participants and 48,084 cases of incident type 2 diabetes. Incidence rate ratios (IRRs) for associations of total fish, shellfish, fatty, lean, fried, freshwater, and saltwater fish intake and type 2 diabetes were derived for each study, adjusting for a consistent set of confounders and combined across studies using random-effects meta-analysis. We stratified all analyses by sex due to observed interaction (&lt;i&gt;p&lt;/i&gt; = 0.002) on the association between fish and type 2 diabetes. In women, for each 100 g/week higher intake the IRRs (95% CIs) of type 2 diabetes were 1.02 (1.01-1.03, &lt;i&gt;I&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 61%) for total fish, 1.04 (1.01-1.07, &lt;i&gt;I&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 46%) for fatty fish, and 1.02 (1.00-1.04, &lt;i&gt;I&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 33%) for lean fish. In men, all associations were null. In women, we observed variation by geographical location: IRRs for total fish were 1.03 (1.02-1.04, &lt;i&gt;I&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 0%) in the Americas and null in other regions. In conclusion, we found evidence of a neutral association between total fish intake and type 2 diabetes in men, but there was a modest positive association among women with heterogeneity across studies, which was partly explained by geographical location and types of fish intake. Future research should investigate the role of cooking methods, accompanying foods and environmental pollutants, but meanwhile, existing dietary regional, national, or international guidelines should continue to guide fish consumption within overall healthy dietary patterns.']", "keywords": {"mesh": [{"term": "Animals", "major": "N"}, {"term": "Confidence Intervals", "major": "N"}, {"term": "Diabetes Mellitus, Type 2", "major": "N"}, {"term": "Feeding Behavior", "major": "Y"}, {"term": "Female", "major": "N"}, {"term": "Fishes", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Incidence", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Prospective Studies", "major": "N"}], "other": ["federated meta-analysis", "fish", "prospective studies", "type 2 diabetes"]}, "full_author_list": ["Pastorino S", "Bishop T", "Sharp S", "Pearce M", "Akbaraly T", "Barbieri N", "Bes-Rastrollo M", "Beulens J", "Chen Z", "Du H", "Duncan B", "Goto A", "H\u00e4rk\u00e4nen T", "Hashemian M", "Kromhout D", "J\u00e4rvinen R", "Kivimaki M", "Knekt P", "Lin X", "Lund E", "Magliano D", "Malekzadeh R", "Mart\u00ednez-Gonz\u00e1lez M", "O'Donoghue G", "O'Gorman D", "Poustchi H", "Rylander C", "Sawada N", "Shaw J", "Schmidt M", "Soedamah-Muthu S", "Sun L", "Wen W", "Wolk A", "Shu X", "Zheng W", "Wareham N", "Forouhi N"], "journal": {"journal_name": "Nutrients", "volume": "13", "issue": "4"}}, {"IDs": {"DOI": "10.3389/fgene.2024.1270387", "PMID": "38348453"}, "title": "Strengths and limitations of non-disclosive data analysis: a comparison of breast cancer survival classifiers using VisualSHIELD.", "abstract": "['Preserving data privacy is an important concern in the research use of patient data. The DataSHIELD suite enables privacy-aware advanced statistical analysis in a federated setting. Despite its many applications, it has a few open practical issues: the complexity of hosting a federated infrastructure, the performance penalty imposed by the privacy-preserving constraints, and the ease of use by non-technical users. In this work, we describe a case study in which we review different breast cancer classifiers and report our findings about the limits and advantages of such non-disclosive suite of tools in a realistic setting. Five independent gene expression datasets of breast cancer survival were downloaded from Gene Expression Omnibus (GEO) and pooled together through the federated infrastructure. Three previously published and two newly proposed 5-year cancer-free survival risk score classifiers were trained in a federated environment, and an additional reference classifier was trained with unconstrained data access. The performance of these six classifiers was systematically evaluated, and the results show that i) the published classifiers do not generalize well when applied to patient cohorts that differ from those used to develop them; ii) among the methods we tried, the classification using logistic regression worked better on average, closely followed by random forest; iii) the unconstrained version of the logistic regression classifier outperformed the federated version by 4&lt;b&gt;%&lt;/b&gt; on average. Reproducibility of our experiments is ensured through the use of VisualSHIELD, an open-source tool that augments DataSHIELD with new functions, a standardized deployment procedure, and a simple graphical user interface.']", "keywords": {"mesh": [], "other": ["computational biology", "federated databases", "non-disclosive data analysis", "transcriptional signatures", "transcriptomic data"]}, "full_author_list": ["Tomasoni D", "Lombardo R", "Lauria M"], "journal": {"journal_name": "Front Genet", "volume": "15", "issue": ""}}, {"IDs": {"DOI": "10.5324/nje.v21i2.1499", "PMID": ""}, "title": "DataSHIELD \u2013 shared individual-level analysis without sharing the data: a biostatistical perspective", "abstract": "Very large sample sizes are required for estimating effects which are known to be small, and for addressing intricate or complex statistical questions. This is often only achievable by pooling data from multiple studies, especially in genetic epidemiology where associations between individual genetic variants and phenotypes of interest are generally weak. However, the physical pooling of experimental data across a consortium is frequently prohibited by the ethico-legal constraints that govern agreements and consents for individual studies. Study level meta-analyses are frequently used so that data from multiple studies need not be pooled to conduct an analysis, though the resulting analysis is necessarily restricted by the available summary statistics. The idea of maintaining data security is also of importance in other areas and approaches to carrying out \u2018secure analyses\u2019 that do not require sharing of data from different sources have been proposed in the technometrics literature. Crucially, the algorithms for fitting certain statistical models can be manipulated so that an individual level meta-analysis can essentially be performed without the need for pooling individual-level data by combining particular summary statistics obtained individually from each study. DataSHIELD (Data Aggregation Through Anonymous Summary-statistics from Harmonised Individual levEL Databases) is a tool to coordinate analyses of data that cannot be pooled. In this paper, we focus on explaining why a DataSHIELD approach yields identical results to an individual level meta-analysis in the case of a generalised linear model, by simply using summary statistics from each study. It is also an efficient approach to carrying out a study level meta-analysis when this is appropriate and when the analysis can be pre-planned. We briefly comment on the IT requirements, together with the ethical and legal challenges which must be addressed.", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Jones E", "Sheehan N", "Masca N", "Wallace S", "Murtagh M", "Burton P"], "journal": {"journal_name": "Norsk Epidemiologi", "volume": "21", "issue": "2"}}, {"IDs": {"DOI": "10.1093/bioinformatics/btad531", "PMID": "37647639"}, "title": "Accessibility of covariance information creates vulnerability in Federated Learning frameworks.", "abstract": "['Federated Learning (FL) is gaining traction in various fields as it enables integrative data analysis without sharing sensitive data, such as in healthcare. However, the risk of data leakage caused by malicious attacks must be considered. In this study, we introduce a novel attack algorithm that relies on being able to compute sample means, sample covariances, and construct known linearly independent vectors on the data owner side.', 'We show that these basic functionalities, which are available in several established FL frameworks, are sufficient to reconstruct privacy-protected data. Additionally, the attack algorithm is robust to defense strategies that involve adding random noise. We demonstrate the limitations of existing frameworks and propose potential defense strategies analyzing the implications of using differential privacy. The novel insights presented in this study will aid in the improvement of FL frameworks.', 'The code examples are provided at GitHub (https://github.com/manuhuth/Data-Leakage-From-Covariances.git). The CNSIM1 dataset, which we used in the manuscript, is available within the DSData R package (https://github.com/datashield/DSData/tree/main/data).']", "keywords": {"mesh": [{"term": "Algorithms", "major": "Y"}, {"term": "Data Analysis", "major": "Y"}, {"term": "Privacy", "major": "N"}], "other": []}, "full_author_list": ["Huth M", "Arruda J", "Gusinow R", "Contento L", "Tacconelli E", "Hasenauer J"], "journal": {"journal_name": "Bioinformatics", "volume": "39", "issue": "9"}}, {"IDs": {"DOI": "10.1002/pst.2415", "PMID": "38973072"}, "title": "Survival Analysis Without Sharing of Individual Patient Data by Using a Gaussian Copula.", "abstract": "['Cox regression and Kaplan-Meier estimations are often needed in clinical research and this requires access to individual patient data (IPD). However, IPD cannot always be shared because of privacy or proprietary restrictions, which complicates the making of such estimations. We propose a method that generates pseudodata replacing the IPD by only sharing non-disclosive aggregates such as IPD marginal moments and a correlation matrix. Such aggregates are collected by a central computer and input as parameters to a Gaussian copula (GC) that generates the pseudodata. Survival inferences are computed on the pseudodata as if it were the IPD. Using practical examples we demonstrate the utility of the method, via the amount of IPD inferential content recoverable by the GC. We compare GC to a summary-based meta-analysis and an IPD bootstrap distributed across several centers. Other pseudodata approaches are also considered. In the empirical results, GC approximates the utility of the IPD bootstrap although it might yield more conservative inferences and it might have limitations in subgroup analyses. Overall, GC avoids many legal problems related to IPD privacy or property while enabling approximation of common IPD survival analyses otherwise difficult to conduct. Sharing more IPD aggregates than is currently practiced could facilitate \"second purpose\"-research and relax concerns regarding IPD access.']", "keywords": {"mesh": [], "other": ["Gaussian\u2010copula", "Kaplan\u2013Meier", "distributed\u2010computing\u2010network", "multilevel\u2010Cox", "multi\u2010center", "privacy"]}, "full_author_list": ["Bonofiglio F"], "journal": {"journal_name": "Pharm Stat", "volume": "", "issue": ""}}, {"IDs": {"DOI": "10.5324/nje.v21i2.1495", "PMID": ""}, "title": "Navigating the perfect [data] storm", "abstract": "Bioscience has recently undergone a series of knowledge-based and technological revolutions. A critical consequence has been increasing recognition of the need to invest in infrastructure. Good access to data (and samples) from multiple studies is axiomatic to the value of this infrastructure. Access must be streamlined, secure, and based upon transparent and \u2018fair\u2019 decision making. It must be clear who has created and who has used which data. Ethico-legal policies and guidelines, which reflect dominant local cultural and societal norms, must take account of the increasingly global nature of bioscience research. A robust data infrastructure must also be attentive to the translational aims and social impact of its knowledge generation. In order to maintain the trust of its constituency \u2013 the general public as well as professional, political, commercial stakeholders \u2013 it must develop mechanisms to take account of all of these perspectives. These considerations form the basis of an emerging data economy. Building on extant achievements and pursuing the ideas outlined here could revolutionise the way we use and manage large-scale data. They have critical implications for biomedical and public health research communities and will be of central relevance for healthcare managers and policy makers, governments and industry. However, if the major challenges are to be met we must continue to invest,both nationally and internationally, in developing the cooperative infrastructures that provide a complementary foil to competitive resourcing mechanisms that drive hypothesis-driven science.", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Murtagh M", "Thorisson G", "Wallace S", "Kaye J", "Demir I", "Fortier I", "Harris J", "Cox D", "Desch\u00eanes M", "Laflamme P", "Ferretti V", "Sheehan N", "Hudson T", "Thomsen A", "Stolk R", "Knoppers B", "Brookes A", "Burton P"], "journal": {"journal_name": "Norsk Epidemiologi", "volume": "21", "issue": "2"}}, {"IDs": {"DOI": "10.1093/bioinformatics/btac616", "PMID": "36073911"}, "title": "dsMTL: a computational framework for privacy-preserving, distributed multi-task machine learning.", "abstract": "['In multi-cohort machine learning studies, it is critical to differentiate between effects that are reproducible across cohorts and those that are cohort-specific. Multi-task learning (MTL) is a machine learning approach that facilitates this differentiation through the simultaneous learning of prediction tasks across cohorts. Since multi-cohort data can often not be combined into a single storage solution, there would be the substantial utility of an MTL application for geographically distributed data sources.', \"Here, we describe the development of 'dsMTL', a computational framework for privacy-preserving, distributed multi-task machine learning that includes three supervised and one unsupervised algorithms. First, we derive the theoretical properties of these methods and the relevant machine learning workflows to ensure the validity of the software implementation. Second, we implement dsMTL as a library for the R programming language, building on the DataSHIELD platform that supports the federated analysis of sensitive individual-level data. Third, we demonstrate the applicability of dsMTL for comorbidity modeling in distributed data. We show that comorbidity modeling using dsMTL outperformed conventional, federated machine learning, as well as the aggregation of multiple models built on the distributed datasets individually. The application of dsMTL was computationally efficient and highly scalable when applied to moderate-size (n\\u2009&lt;\\u2009500), real expression data given the actual network latency.\", 'dsMTL is freely available at https://github.com/transbioZI/dsMTLBase (server-side package) and https://github.com/transbioZI/dsMTLClient (client-side package).', 'Supplementary data are available at Bioinformatics online.']", "keywords": {"mesh": [{"term": "Humans", "major": "N"}, {"term": "Privacy", "major": "Y"}, {"term": "Machine Learning", "major": "Y"}, {"term": "Software", "major": "N"}, {"term": "Programming Languages", "major": "N"}, {"term": "Algorithms", "major": "N"}], "other": []}, "full_author_list": ["Cao H", "Zhang Y", "Baumbach J", "Burton P", "Dwyer D", "Koutsouleris N", "Matschinske J", "Marcon Y", "Rajan S", "Rieg T", "Ryser-Welch P", "Sp\u00e4th J", "Herrmann C", "Schwarz E"], "journal": {"journal_name": "Bioinformatics", "volume": "38", "issue": "21"}}, {"IDs": {"DOI": "10.3233/978-1-61499-959-1-115", "PMID": "30942726"}, "title": "A Queue-Poll Extension and DataSHIELD: Standardised, Monitored, Indirect and Secure Access to Sensitive Data.", "abstract": "['Analyzing data across hospitals and institutions without the data leaving the hospitals and adding institutions to a trusted network is an important part of privacy preserving data analysis. This work implements a queue-poll extension and integrates with DataSHIELD to allow for a standardized, monitored, indirect and secure access to data. The extension was created using the HTTP protocol and requests are not pushed into a participating institution but are sent to a server outside an institutional network. These requests are then pulled into the institution from within, executed and the response sent back to the outside server, which relays the request back to the request sender. We found that the requests were slower than a direct push request, but also that the integration of new institutions into the network was easily achieved. We propose that future work should focus on optimizing the monitoring and speed of the service. The service created here could reduce the barriers to entry for institutions to form an analysis network and can be used not only to drive analysis but also the sharing of resulting information and models.']", "keywords": {"mesh": [{"term": "Computer Systems", "major": "N"}, {"term": "Computers", "major": "N"}, {"term": "Confidentiality", "major": "Y"}, {"term": "Privacy", "major": "N"}, {"term": "Software", "major": "Y"}], "other": ["Distributed privacy preserving analysis", "Machine-learning", "data analysis network", "data governance"]}, "full_author_list": ["Gruendner J", "Prokosch H", "Schindler S", "Lenz S", "Binder H"], "journal": {"journal_name": "Stud Health Technol Inform", "volume": "258", "issue": ""}}, {"IDs": {"DOI": "10.1016/j.envint.2024.108453", "PMID": "38368715"}, "title": "Urban environment in pregnancy and postpartum depression: An individual participant data meta-analysis of 12 European birth cohorts.", "abstract": "['Urban environmental exposures associate with adult depression, but it is unclear whether they are associated to postpartum depression (PPD).', 'We investigated associations between urban environment exposures during pregnancy and PPD.', 'We included women with singleton deliveries to liveborn children from 12 European birth cohorts (N with minimum one exposure\\xa0=\\xa030,772, analysis N range 17,686-30,716 depending on exposure; representing 26-46\\xa0% of the 66,825 eligible women). We estimated maternal exposure during pregnancy to ambient air pollution with nitrogen dioxide (NO&lt;sub&gt;2&lt;/sub&gt;) and particulate matter (PM&lt;sub&gt;2.5&lt;/sub&gt; and PM&lt;sub&gt;10&lt;/sub&gt;), road traffic noise (L&lt;sub&gt;den&lt;/sub&gt;), natural spaces (Normalised Difference Vegetation Index; NDVI, proximity to major green or blue spaces) and built environment (population density, facility richness and walkability). Maternal PPD was assessed 3-18\\xa0months after birth using self-completed questionnaires. We used adjusted logistic regression models to estimate cohort-specific associations between each exposure and PPD and combined results via meta-analysis using DataSHIELD.', 'Of the 30,772 women included, 3,078 (10\\xa0%) reported having PPD. Exposure to PM&lt;sub&gt;10&lt;/sub&gt; was associated with slightly increased odds of PPD (adjusted odd ratios (OR) of 1.08 [95\\xa0% Confidence Intervals (CI): 0.99, 1.17] per inter quartile range increment of PM&lt;sub&gt;10&lt;/sub&gt;) whilst associations for exposure to NO&lt;sub&gt;2&lt;/sub&gt; and PM&lt;sub&gt;2.5&lt;/sub&gt; were close to null. Exposure to high levels of road traffic noise (\u226565\\xa0dB vs.\\xa0&lt;\\xa065\\xa0dB) was associated with an OR of 1.12 [CI: 0.95, 1.32]. Associations between green spaces and PPD were close to null; whilst proximity to major blue spaces was associated with increased risk of PPD (OR 1.12, 95\\xa0%CI: 1.00, 1.26). All associations between built environment and PPD were close to null. Multiple exposure models showed similar results.', 'The study findings suggest that exposure to PM&lt;sub&gt;10,&lt;/sub&gt; road traffic noise and blue spaces in pregnancy may increase PPD risk, however future studies should explore this causally.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Pregnancy", "major": "N"}, {"term": "Air Pollutants", "major": "Y"}, {"term": "Air Pollution", "major": "Y"}, {"term": "Birth Cohort", "major": "N"}, {"term": "Depression, Postpartum", "major": "Y"}, {"term": "Environmental Exposure", "major": "N"}, {"term": "Nitrogen Dioxide", "major": "N"}, {"term": "Particulate Matter", "major": "N"}, {"term": "Infant, Newborn", "major": "N"}], "other": ["Ambient air pollution", "Built environment", "Natural spaces", "Postpartum depression", "Traffic noise", "Urban"]}, "full_author_list": ["Cadman T", "Strandberg-Larsen K", "Calas L", "Christiansen M", "Culpin I", "Dadvand P", "de Castro M", "Foraster M", "Fossati S", "Guxens M", "Harris J", "Hillegers M", "Jaddoe V", "Lee Y", "Lepeule J", "El Marroun H", "Maule M", "McEachen R", "Moccia C", "Nader J", "Nieuwenhuijsen M", "Nybo Andersen A", "Pearson R", "Swertz M", "Vafeiadi M", "Vrijheid M", "Wright J", "Lawlor D", "Pedersen M"], "journal": {"journal_name": "Environ Int", "volume": "185", "issue": ""}}, {"IDs": {"DOI": "10.1097/EE9.0000000000000166", "PMID": "34934888"}, "title": "Advancing tools for human early lifecourse exposome research and translation (ATHLETE): Project overview.", "abstract": "[\"Early life stages are vulnerable to environmental hazards and present important windows of opportunity for lifelong disease prevention. This makes early life a relevant starting point for exposome studies. The Advancing Tools for Human Early Lifecourse Exposome Research and Translation (ATHLETE) project aims to develop a toolbox of exposome tools and a Europe-wide exposome cohort that will be used to systematically quantify the effects of a wide range of community- and individual-level environmental risk factors on mental, cardiometabolic, and respiratory health outcomes and associated biological pathways, longitudinally from early pregnancy through to adolescence. Exposome tool and data development include as follows: (1) a findable, accessible, interoperable, reusable (FAIR) data infrastructure for early life exposome cohort data, including 16 prospective birth cohorts in 11 European countries; (2) targeted and nontargeted approaches to measure a wide range of environmental exposures (urban, chemical, physical, behavioral, social); (3) advanced statistical and toxicological strategies to analyze complex multidimensional exposome data; (4) estimation of associations between the exposome and early organ development, health trajectories, and biological (metagenomic, metabolomic, epigenetic, aging, and stress) pathways; (5) intervention strategies to improve early life urban and chemical exposomes, co-produced with local communities; and (6) child health impacts and associated costs related to the exposome. Data, tools, and results will be assembled in an openly accessible toolbox, which will provide great opportunities for researchers, policymakers, and other stakeholders, beyond the duration of the project. ATHLETE's results will help to better understand and prevent health damage from environmental exposures and their mixtures from the earliest parts of the life course onward.\"]", "keywords": {"mesh": [], "other": ["Adolescent health", "Child health", "Early life", "Exposome", "Exposure assessment"]}, "full_author_list": ["Vrijheid M", "Basaga\u00f1a X", "Gonzalez J", "Jaddoe V", "Jensen G", "Keun H", "McEachan R", "Porcel J", "Siroux V", "Swertz M", "Thomsen C", "Aasvang G", "Andru\u0161aityt\u0117 S", "Angeli K", "Avraam D", "Ballester F", "Burton P", "Bustamante M", "Casas M", "Chatzi L", "Chevrier C", "Cingotti N", "Conti D", "Cr\u00e9pet A", "Dadvand P", "Duijts L", "van Enckevort E", "Esplugues A", "Fossati S", "Garlantezec R", "G\u00f3mez Roig M", "Grazuleviciene R", "G\u00fctzkow K", "Guxens M", "Haakma S", "Hessel E", "Hoyles L", "Hyde E", "Klanova J", "van Klaveren J", "Kortenkamp A", "Le Brusquet L", "Leenen I", "Lertxundi A", "Lertxundi N", "Lionis C", "Llop S", "Lopez-Espinosa M", "Lyon-Caen S", "Maitre L", "Mason D", "Mathy S", "Mazarico E", "Nawrot T", "Nieuwenhuijsen M", "Ortiz R", "Pedersen M", "Perell\u00f3 J", "P\u00e9rez-Cruz M", "Philippat C", "Piler P", "Pizzi C", "Quentin J", "Richiardi L", "Rodriguez A", "Roumeliotaki T", "Sabin Capote J", "Santiago L", "Santos S", "Siskos A", "Strandberg-Larsen K", "Stratakis N", "Sunyer J", "Tenenhaus A", "Vafeiadi M", "Wilson R", "Wright J", "Yang T", "Slama R"], "journal": {"journal_name": "Environ Epidemiol", "volume": "5", "issue": "5"}}, {"IDs": {"DOI": "10.1016/j.envres.2016.09.014", "PMID": "27692672"}, "title": "Road traffic noise, blood pressure and heart rate: Pooled analyses of harmonized data from 88,336 participants.", "abstract": "['Exposure to road traffic noise may increase blood pressure and heart rate. It is unclear to what extent exposure to air pollution may influence this relationship. We investigated associations between noise, blood pressure and heart rate, with harmonized data from three European cohorts, while taking into account exposure to air pollution.', 'Road traffic noise exposure was assessed using a European noise model based on the Common Noise Assessment Methods in Europe framework (CNOSSOS-EU). Exposure to air pollution was estimated using a European-wide land use regression model. Blood pressure and heart rate were obtained by trained clinical professionals. Pooled cross-sectional analyses of harmonized data were conducted at the individual level and with random-effects meta-analyses.', 'We analyzed data from 88,336 participants, across the three participating cohorts (mean age 47.0 (\u00b113.9) years). Each 10dB(A) increase in noise was associated with a 0.93 (95% CI 0.76;1.11) bpm increase in heart rate, but with a decrease in blood pressure of 0.01 (95% CI -0.24;0.23) mmHg for systolic and 0.38 (95% CI -0.53; -0.24) mmHg for diastolic blood pressure. Adjustments for PM&lt;sub&gt;10&lt;/sub&gt; or NO&lt;sub&gt;2&lt;/sub&gt; attenuated the associations, but remained significant for DBP and HR. Results for BP differed by cohort, with negative associations with noise in LifeLines, no significant associations in EPIC-Oxford, and positive associations with noise &gt;60dB(A) in HUNT3.', 'Our study suggests that road traffic noise may be related to increased heart rate. No consistent evidence for a relation between noise and blood pressure was found.']", "keywords": {"mesh": [{"term": "Adolescent", "major": "N"}, {"term": "Adult", "major": "N"}, {"term": "Aged", "major": "N"}, {"term": "Aged, 80 and over", "major": "N"}, {"term": "Air Pollution", "major": "N"}, {"term": "Blood Pressure", "major": "N"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Cross-Sectional Studies", "major": "N"}, {"term": "Europe", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Heart Rate", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Meta-Analysis as Topic", "major": "N"}, {"term": "Middle Aged", "major": "N"}, {"term": "Noise, Transportation", "major": "N"}, {"term": "Young Adult", "major": "N"}], "other": ["Air pollution", "Blood pressure", "Epidemiology", "Heart rate", "Meta-analysis", "Noise"]}, "full_author_list": ["Zijlema W", "Cai Y", "Doiron D", "Mbatchou S", "Fortier I", "Gulliver J", "de Hoogh K", "Morley D", "Hodgson S", "Elliott P", "Key T", "Kongsgard H", "Hveem K", "Gaye A", "Burton P", "Hansell A", "Stolk R", "Rosmalen J"], "journal": {"journal_name": "Environ Res", "volume": "151", "issue": ""}}, {"IDs": {"DOI": "10.1111/1471-0528.15476", "PMID": "30230190"}, "title": "Associations between maternal physical activity in early and late pregnancy and offspring birth size: remote federated individual level meta-analysis from eight cohort studies.", "abstract": "['Evidence on the impact of leisure time physical activity (LTPA) in pregnancy on birth size is inconsistent. We aimed to examine the association between LTPA during early and late pregnancy and newborn anthropometric outcomes.', 'Individual level meta-analysis, which reduces heterogeneity across studies.', 'A consortium of eight population-based studies (seven European and one US) comprising 72\\xa0694 participants.', 'Generalised linear models with consistent inclusion of confounders (gestational age, sex, parity, maternal age, education, ethnicity, BMI, smoking, and alcohol intake) were used to test associations between self-reported LTPA at either early (8-18\\xa0weeks gestation) or late pregnancy (30+\\xa0weeks) and the outcomes. Results were pooled using random effects meta-analyses.', 'Birth weight, large-for-gestational age (LGA), macrosomia, small-for-gestational age (SGA), % body fat, and ponderal index at birth.', 'Late, but not early, gestation maternal moderate to vigorous physical activity (MVPA), vigorous activity, and LTPA energy expenditure were modestly inversely associated with BW, LGA, macrosomia, and ponderal index, without heterogeneity (all: I&lt;sup&gt;2&lt;/sup&gt; \\xa0=\\xa00%). For each extra hour/week of MVPA, RR for LGA and macrosomia were 0.97 (95% CI: 0.96, 0.98) and 0.96 (95% CI: 0.94, 0.98), respectively. Associations were only modestly reduced\\xa0after additional adjustments for maternal BMI and gestational diabetes. No measure of LTPA was associated with risk for SGA.', 'Physical activity in late, but not early, pregnancy is consistently associated with modestly lower risk of LGA and macrosomia, but not SGA.', 'In an individual participant meta-analysis, late pregnancy moderate to vigorous physical activity modestly reduced birth size outcomes.']", "keywords": {"mesh": [{"term": "Adipose Tissue", "major": "N"}, {"term": "Adult", "major": "N"}, {"term": "Birth Weight", "major": "Y"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Diabetes, Gestational", "major": "N"}, {"term": "Energy Metabolism", "major": "N"}, {"term": "Exercise", "major": "Y"}, {"term": "Female", "major": "N"}, {"term": "Fetal Macrosomia", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Infant, Newborn", "major": "N"}, {"term": "Infant, Small for Gestational Age", "major": "Y"}, {"term": "Linear Models", "major": "N"}, {"term": "Obesity", "major": "N"}, {"term": "Overweight", "major": "N"}, {"term": "Pregnancy", "major": "N"}, {"term": "Pregnancy Complications", "major": "N"}, {"term": "Pregnancy Trimester, First", "major": "N"}, {"term": "Pregnancy Trimester, Second", "major": "N"}, {"term": "Pregnancy Trimester, Third", "major": "N"}, {"term": "Protective Factors", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Young Adult", "major": "N"}], "other": ["Birth weight", "large-for-gestational age", "macrosomia", "physical activity", "pregnancy", "small-for-gestational age"]}, "full_author_list": ["Pastorino S", "Bishop T", "Crozier S", "Granstr\u00f6m C", "Kordas K", "K\u00fcpers L", "O'Brien E", "Polanska K", "Sauder K", "Zafarmand M", "Wilson R", "Agyemang C", "Burton P", "Cooper C", "Corpeleijn E", "Dabelea D", "Hanke W", "Inskip H", "McAuliffe F", "Olsen S", "Vrijkotte T", "Brage S", "Kennedy A", "O'Gorman D", "Scherer P", "Wijndaele K", "Wareham N", "Desoye G", "Ong K"], "journal": {"journal_name": "BJOG", "volume": "126", "issue": "4"}}, {"IDs": {"DOI": "10.1080/14636778.2013.846582", "PMID": ""}, "title": "Data sharing across biobanks: epistemic values, data mutability and data incommensurability", "abstract": "Despite the centrality of epistemic issues in biobank knowledge generation, there is currently a lacuna in research addressing the epistemic assumptions of biobank science and data sharing. The article addresses this lacuna. Using the insights of philosophical and sociological analysis, we examine standardization and harmonization and central challenges biobank data-sharing faces. We defend two central epistemic values, namely \u201cspatial and temporal translatability\u201d and \u201cepistemic adequacy\u201d which foster effective biobank knowledge generation. The first refers to the way in which biobank data need to be re-usable and re-purposable by bioscience researchers who did not create them. Given the perennial issues of data mutability and incommensurability, we also propose \u201cepistemic adequacy.\u201d In uncovering the social and epistemic foundations of biobank science, we emphasize issues essential for achieving effective and transparent biobank practice and productive communication and engagement with the public about the nature, potential and limits of biobanks.", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Demir I", "Murtagh M"], "journal": {"journal_name": "New Genetics and Society", "volume": "32", "issue": "4"}}, {"IDs": {"DOI": "10.1089/bio.2015.0119", "PMID": "27219861"}, "title": "What Does Anonymization Mean? DataSHIELD and the Need for Consensus on Anonymization Terminology.", "abstract": "[\"Anonymization is a recognized process by which identifiers can be removed from identifiable data to protect an individual's confidentiality and is used as a standard practice when sharing data in biomedical research. However, a plethora of terms, such as coding, pseudonymization, unlinked, and deidentified, have been and continue to be used, leading to confusion and uncertainty. This article shows that this is a historic problem and argues that such continuing uncertainty regarding the levels of protection given to data risks damaging initiatives designed to assist researchers conducting cross-national studies and sharing data internationally. DataSHIELD and the creation of a legal template are used as examples of initiatives that rely on anonymization, but where the inconsistency in terminology could hinder progress. More broadly, this article argues that there is a real possibility that there could be possible damage to the public's trust in research and the institutions that carry it out by relying on vague notions of the anonymization process. Research participants whose lack of clear understanding of the research process is compensated for by trusting those carrying out the research may have that trust damaged if the level of protection given to their data does not match their expectations. One step toward ensuring understanding between parties would be consistent use of clearly defined terminology used internationally, so that all those involved are clear on the level of identifiability of any particular set of data and, therefore, how that data can be accessed and shared.\"]", "keywords": {"mesh": [{"term": "Computer Security", "major": "N"}, {"term": "Consensus", "major": "N"}, {"term": "Data Anonymization", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Dissemination", "major": "N"}, {"term": "Terminology as Topic", "major": "N"}], "other": []}, "full_author_list": ["Wallace S"], "journal": {"journal_name": "Biopreserv Biobank", "volume": "14", "issue": "3"}}, {"IDs": {"DOI": "10.1080/01621459.2021.1904958", "PMID": "37975021"}, "title": "Individual Data Protected Integrative Regression Analysis of High-Dimensional Heterogeneous Data.", "abstract": "['Evidence-based decision making often relies on meta-analyzing multiple studies, which enables more precise estimation and investigation of generalizability. Integrative analysis of multiple heterogeneous studies is, however, highly challenging in the ultra high-dimensional setting. The challenge is even more pronounced when the individual-level data cannot be shared across studies, known as DataSHIELD contraint. Under sparse regression models that are assumed to be similar yet not identical across studies, we propose in this paper a novel integrative estimation procedure for data-Shielding High-dimensional Integrative Regression (SHIR). SHIR protects individual data through summary-statistics-based integrating procedure, accommodates between-study heterogeneity in both the covariate distribution and model parameters, and attains consistent variable selection. Theoretically, SHIR is statistically more efficient than the existing distributed approaches that integrate debiased LASSO estimators from the local sites. Furthermore, the estimation error incurred by aggregating derived data is negligible compared to the statistical minimax rate and SHIR is shown to be asymptotically equivalent in estimation to the ideal estimator obtained by sharing all data. The finite-sample performance of our method is studied and compared with existing approaches via extensive simulation settings. We further illustrate the utility of SHIR to derive phenotyping algorithms for coronary artery disease using electronic health records data from multiple chronic disease cohorts.']", "keywords": {"mesh": [], "other": ["DataSHIELD", "Distributed learning", "High dimensionality", "Model heterogeneity", "Rate optimality", "Sparsistency"]}, "full_author_list": ["Cai T", "Liu M", "Xia Y"], "journal": {"journal_name": "J Am Stat Assoc", "volume": "117", "issue": "540"}}, {"IDs": {"DOI": "10.1089/bio.2016.0002", "PMID": "27200470"}, "title": "International Data Sharing in Practice: New Technologies Meet Old Governance.", "abstract": "['The social structures that govern data/sample release aim to safeguard the confidentiality and privacy of cohort research participants (without whom there would be no data or samples) and enable the realization of societal benefit through optimizing the scientific use of those cohorts. Within collaborations involving multiple cohorts and biobanks, however, the local, national, and supranational institutional and legal guidelines for research (which produce a multiplicity of data access governance structures and guidelines) risk impeding the very science that is the raison d\\'etre of these consortia. We present an ethnographic study, which examined the epistemic and nonepistemic values driving decisions about data access and their consequences in the context of the pilot of an integrated approach to co-analysis of data. We demonstrate how the potential analytic flexibility offered by this approach was lost under contemporary data access governance. We identify three dominant values: protecting the research participant, protecting the study, and protecting the researcher. These values were both supported by and juxtaposed against a \"public good\" argument, and each was used as a rationale to both promote and inhibit sharing of data. While protection of the research participants was central to access permissions, decisions were also attentive to the desire of researchers to see their efforts in building population biobanks and cohorts realized in the form of scientific outputs. We conclude that systems for governing and enabling data access in large consortia need to (1) protect disclosure of research participant information or identity, (2) ensure the specific expectations of research participants are met, (3) embody systems of review that are transparent and not compromised by the specific interests of one particular group of stakeholders, and (4) facilitate data access procedures that are timely and efficient. Practical solutions are urgently needed. New approaches to data access governance should be trialed (and formally evaluated) with input from and discussion with stakeholders.']", "keywords": {"mesh": [{"term": "Biological Specimen Banks", "major": "N"}, {"term": "Biomedical Research", "major": "N"}, {"term": "Confidentiality", "major": "N"}, {"term": "Disclosure", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Dissemination", "major": "N"}, {"term": "International Cooperation", "major": "N"}, {"term": "Research Personnel", "major": "N"}], "other": []}, "full_author_list": ["Murtagh M", "Turner A", "Minion J", "Fay M", "Burton P"], "journal": {"journal_name": "Biopreserv Biobank", "volume": "14", "issue": "3"}}, {"IDs": {"DOI": "10.5334/dsj-2017-021", "PMID": ""}, "title": "DataSHIELD \u2013 New Directions and Dimensions", "abstract": "In disciplines such as biomedicine and social sciences, sharing and combining sensitive individual-level data is often prohibited by ethical-legal or governance constraints and other barriers such as the control of intellectual property or the huge sample sizes. DataSHIELD (Data Aggregation Through Anonymous Summary-statistics from Harmonised Individual-levEL Databases) is a distributed approach that allows the analysis of sensitive individual-level data from one study, and the co-analysis of such data from several studies simultaneously without physically pooling them or disclosing any data.\n\nFollowing initial proof of principle, a stable DataSHIELD platform has now been implemented in a number of epidemiological consortia. This paper reports three new applications of DataSHIELD including application to post-publication sensitive data analysis, text data analysis and privacy protected data visualisation. Expansion of DataSHIELD analytic functionality and application to additional data types demonstrate the broad applications of the software beyond biomedical sciences.", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Wilson R", "Butters O", "Avraam D", "Baker J", "Tedds J", "Turner A", "Murtagh M", "Burton P"], "journal": {"journal_name": "Data Science Journal", "volume": "16", "issue": ""}}, {"IDs": {"DOI": "10.12688/wellcomeopenres.12441.1", "PMID": "28989981"}, "title": "Synthetic ALSPAC longitudinal datasets for the Big Data VR project.", "abstract": "['Three synthetic datasets - of observation size 15,000, 155,000 and 1,555,000 participants, respectively - were created by simulating eleven cardiac and anthropometric variables from nine collection ages of the ALSAPC birth cohort study. The synthetic datasets retain similar data properties to the ALSPAC study data they are simulated from (co-variance matrices, as well as the mean and variance values of the variables) without including the original data itself or disclosing participant information. \\xa0In this instance, the three synthetic datasets have been utilised in an academia-industry collaboration to build a prototype virtual reality data analysis software, but they could have a broader use in method and software development projects where sensitive data cannot be freely shared.']", "keywords": {"mesh": [], "other": ["ALSPAC", "Simulated data", "data visualisation", "synthetic data", "virtual reality", "visual analytics"]}, "full_author_list": ["Avraam D", "Wilson R", "Burton P"], "journal": {"journal_name": "Wellcome Open Res", "volume": "2", "issue": ""}}, {"IDs": {"DOI": "10.1002/sim.8470", "PMID": "31944335"}, "title": "Recovery of original individual person data (IPD) inferences from empirical IPD summaries only: Applications to distributed computing under disclosure constraints.", "abstract": "['There are many settings where individual person data (IPD) are not available, due to privacy or technical reasons, and one must work with IPD proxies, such as summary statistics, to approximate original IPD inferences, that is, the results of statistical analyses that would ideally have been performed on individual-level data. For instance, in a distributed computing setting, as implemented in the DataSHIELD software framework, different centers can only share IPD proxies to obtain pooled IPD inferences. Such privacy requirements limit the scope of statistical investigation. For example, it can be challenging to perform between-center random-effect regression models. To increase modeling freedom we propose a method that only uses simple nondisclosive summaries of the original IPD as input, such as empirical marginal moments and correlation matrices, and generates artificial data compatible with those summary features. Specifically, data are generated from a Gaussian copula with marginal and joint components specified by the above summaries. The goal is to reproduce original IPD features in the artificial data, such that original IPD inferences are recovered from the artificial data. In an application example, and through simulations, we show that we can recover estimates of a multivariable IPD random-effect logistic regression, from artificial data generated via the Gaussian copula using the above IPD summaries, suggesting the proposed approach provides a generally applicable strategy for distributed computing settings with data protection constraints.']", "keywords": {"mesh": [{"term": "Computer Security", "major": "N"}, {"term": "Data Interpretation, Statistical", "major": "N"}, {"term": "Disclosure", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Logistic Models", "major": "N"}, {"term": "Research Design", "major": "Y"}], "other": ["DataSHIELD", "GLM", "copula", "multivariable", "random-effect"]}, "full_author_list": ["Bonofiglio F", "Schumacher M", "Binder H"], "journal": {"journal_name": "Stat Med", "volume": "39", "issue": "8"}}, {"IDs": {"DOI": "10.1093/eurheartj/ehx263", "PMID": "28575405"}, "title": "Long-term exposure to road traffic noise, ambient air pollution, and cardiovascular risk factors in the HUNT and lifelines cohorts.", "abstract": "['Blood biochemistry may provide information on associations between road traffic noise, air pollution, and cardiovascular disease risk. We evaluated this in two large European cohorts (HUNT3, Lifelines).', 'Road traffic noise exposure was modelled for 2009 using a simplified version of the Common Noise Assessment Methods in Europe (CNOSSOS-EU). Annual ambient air pollution (PM10, NO2) at residence was estimated for 2007 using a Land Use Regression model. The statistical platform DataSHIELD was used to pool data from 144\\u2009082 participants aged\\u2009\u226520\\u2009years to enable individual-level analysis. Generalized linear models were fitted to assess cross-sectional associations between pollutants and high-sensitivity C-reactive protein (hsCRP), blood lipids and for (Lifelines only) fasting blood glucose, for samples taken during recruitment in 2006-2013. Pooling both cohorts, an inter-quartile range (IQR) higher day-time noise (5.1\\u2009dB(A)) was associated with 1.1% [95% confidence interval (95% CI: 0.02-2.2%)] higher hsCRP, 0.7% (95% CI: 0.3-1.1%) higher triglycerides, and 0.5% (95% CI: 0.3-0.7%) higher high-density lipoprotein (HDL); only the association with HDL was robust to adjustment for air pollution. An IQR higher PM10 (2.0 \u00b5g/m3) or NO2 (7.4 \u00b5g/m3) was associated with higher triglycerides (1.9%, 95% CI: 1.5-2.4% and 2.2%, 95% CI: 1.6-2.7%), independent of adjustment for noise. Additionally for NO2, a significant association with hsCRP (1.9%, 95% CI: 0.5-3.3%) was seen. In Lifelines, an IQR higher noise (4.2\\u2009dB(A)) and PM10 (2.4 \u00b5g/m3) was associated with 0.2% (95% CI: 0.1-0.3%) and 0.6% (95% CI: 0.4-0.7%) higher fasting glucose respectively, with both remaining robust to adjustment for air/noise pollution.', 'Long-term exposures to road traffic noise and ambient air pollution were associated with blood biochemistry, providing a possible link between road traffic noise/air pollution and cardio-metabolic disease risk.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Air Pollutants", "major": "N"}, {"term": "Air Pollution", "major": "N"}, {"term": "Cardiovascular Diseases", "major": "N"}, {"term": "Environmental Exposure", "major": "N"}, {"term": "Europe", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Middle Aged", "major": "N"}, {"term": "Noise, Transportation", "major": "N"}, {"term": "Prospective Studies", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Socioeconomic Factors", "major": "N"}], "other": ["Air pollution", "Blood glucose", "Blood lipids", "Systemic inflammation", "Traffic noise"]}, "full_author_list": ["Cai Y", "Hansell A", "Blangiardo M", "Burton P", "de Hoogh K", "Doiron D", "Fortier I", "Gulliver J", "Hveem K", "Mbatchou S", "Morley D", "Stolk R", "Zijlema W", "Elliott P", "Hodgson S"], "journal": {"journal_name": "Eur Heart J", "volume": "38", "issue": "29"}}, {"IDs": {"DOI": "10.1186/s12931-023-02607-y", "PMID": "38238846"}, "title": "Alpha-1-antitrypsin-deficiency is associated with lower cardiovascular risk: an approach based on federated learning.", "abstract": "['Chronic obstructive pulmonary disease (COPD) is an inflammatory multisystemic disease caused by environmental exposures and/or genetic factors. Inherited alpha-1-antitrypsin deficiency (AATD) is one of the best recognized genetic factors increasing the risk for an early onset COPD with emphysema. The aim of this study was to gain a better understanding of the associations between comorbidities and specific biomarkers in COPD patients with and without AATD to enable future investigations aimed, for example, at identifying risk factors or improving care.', 'We focused on cardiovascular comorbidities, blood high sensitivity troponin (hs-troponin) and lipid profiles in COPD patients with and without AATD. We used clinical data from six German University Medical Centres of the MIRACUM (Medical Informatics Initiative in Research and Medicine) consortium. The codes for the international classification of diseases (ICD) were used for COPD as a main diagnosis and for comorbidities and blood laboratory data were obtained. Data analyses were based on the DataSHIELD framework.', 'Out of 112,852 visits complete information was available for 43,057 COPD patients. According to our findings, 746 patients with AATD (1.73%) showed significantly lower total blood cholesterol levels and less cardiovascular comorbidities than non-AATD COPD patients. Moreover, after adjusting for the confounder factors, such as age, gender, and nicotine abuse, we confirmed that hs-troponin is a suitable predictor of overall mortality in COPD patients. The comorbidities associated with AATD in the current study differ from other studies, which may reflect geographic and population-based differences as well as the heterogeneous characteristics of AATD.', 'The concept of MIRACUM is suitable for the analysis of a large healthcare database. This study provided evidence that COPD patients with AATD have a lower cardiovascular risk and revealed that hs-troponin is a predictor for hospital mortality in individuals with COPD.']", "keywords": {"mesh": [{"term": "Humans", "major": "N"}, {"term": "alpha 1-Antitrypsin Deficiency", "major": "Y"}, {"term": "Cardiovascular Diseases", "major": "Y"}, {"term": "Heart Disease Risk Factors", "major": "N"}, {"term": "Pulmonary Disease, Chronic Obstructive", "major": "Y"}, {"term": "Risk Factors", "major": "N"}, {"term": "Troponin", "major": "N"}], "other": ["Alpha-1-antitrypsin deficiency", "COPD", "Cardiovascular", "Cholesterol", "Troponin"]}, "full_author_list": ["Z\u00f6ller D", "Haverkamp C", "Makoudjou A", "Sofack G", "Kiefer S", "Gebele D", "Pfaffenlehner M", "Boeker M", "Binder H", "Karki K", "Seidemann C", "Schmeck B", "Greulich T", "Renz H", "Schild S", "Seuchter S", "Tibyampansha D", "Buhl R", "Rohde G", "Trudzinski F", "Bals R", "Janciauskiene S", "Stolz D", "F\u00e4hndrich S"], "journal": {"journal_name": "Respir Res", "volume": "25", "issue": "1"}}, {"IDs": {"DOI": "10.1093/jn/nxab077", "PMID": "33847346"}, "title": "Dietary Macronutrient Composition in Relation to Circulating HDL and Non-HDL Cholesterol: A Federated Individual-Level Analysis of Cross-Sectional Data from Adolescents and Adults in 8 European Studies.", "abstract": "['Associations between increased dietary fat and decreased carbohydrate intake with circulating HDL and non-HDL cholesterol have not been conclusively determined.', 'We assessed these relations in 8 European observational human studies participating in the European Nutritional Phenotype Assessment and Data Sharing Initiative (ENPADASI) using harmonized data.', 'Dietary macronutrient intake was recorded using study-specific dietary assessment tools. Main outcome measures were lipoprotein cholesterol concentrations: HDL cholesterol (mg/dL) and non-HDL cholesterol (mg/dL). A cross-sectional analysis on 5919 participants (54% female) aged 13-80 y was undertaken using the statistical platform DataSHIELD that allows remote/federated nondisclosive analysis of individual-level data. Generalized linear models (GLM) were fitted to assess associations between replacing 5% of energy from carbohydrates with equivalent energy from total fats, SFAs, MUFAs, or PUFAs with circulating HDL cholesterol and non-HDL cholesterol. GLM were adjusted for study source, age, sex, smoking status, alcohol intake and BMI.', 'The replacement of 5% of energy from carbohydrates with total fats or MUFAs was statistically significantly associated with 0.67\\xa0mg/dL (95% CI: 0.40, 0.94) or 0.99\\xa0mg/dL (95% CI: 0.37, 1.60) higher HDL cholesterol, respectively, but not with non-HDL cholesterol concentrations. The replacement of 5% of energy from carbohydrates with SFAs or PUFAs was not associated with HDL cholesterol, but SFAs were statistically significantly associated with 1.94\\xa0mg/dL (95% CI: 0.08, 3.79) higher non-HDL cholesterol, and PUFAs with -3.91\\xa0mg/dL (95% CI: -6.98, -0.84) lower non-HDL cholesterol concentrations. A statistically significant interaction by sex for the association of replacing carbohydrates with MUFAs and non-HDL cholesterol was observed, showing a statistically significant inverse association in males and no statistically significant association in females. We observed no statistically significant interaction by age.', 'The replacement of dietary carbohydrates with fats had favorable effects on lipoprotein cholesterol concentrations in European adolescents and adults when fats were consumed as MUFAs or PUFAs but not as SFAs.']", "keywords": {"mesh": [{"term": "Adolescent", "major": "N"}, {"term": "Cholesterol, HDL", "major": "N"}, {"term": "Cross-Sectional Studies", "major": "N"}, {"term": "Diet", "major": "N"}, {"term": "Dietary Fats", "major": "Y"}, {"term": "Fatty Acids", "major": "Y"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Nutrients", "major": "N"}, {"term": "Observational Studies as Topic", "major": "N"}], "other": ["adolescents", "adults", "blood lipids", "carbohydrates", "data integration", "data sharing", "dietary intake", "energy density models", "fatty acids", "substitution"]}, "full_author_list": ["Pinart M", "Jeran S", "Boeing H", "Stelmach-Mardas M", "Standl M", "Schulz H", "Harris C", "von Berg A", "Herberth G", "Koletzko S", "Linseisen J", "Breuninger T", "N\u00f6thlings U", "Barbaresko J", "Benda S", "Lachat C", "Yang C", "Gasparini P", "Robino A", "Rojo-Mart\u00ednez G", "Casta\u00f1o L", "Guillaume M", "Donneau A", "Hoge A", "Gillain N", "Avraam D", "Burton P", "Bouwman J", "Pischon T", "Nimptsch K"], "journal": {"journal_name": "J Nutr", "volume": "151", "issue": "8"}}, {"IDs": {"DOI": "10.1289/EHP1353", "PMID": "28963089"}, "title": "Residential Air Pollution and Associations with Wheeze and Shortness of Breath in Adults: A Combined Analysis of Cross-Sectional Data from Two Large European Cohorts.", "abstract": "['Research examining associations between air pollution exposure and respiratory symptoms in adults has generally been inconclusive. This may be related in part to sample size issues, which also preclude analysis in potentially vulnerable subgroups.', 'We estimated associations between air pollution exposures and the prevalence of wheeze and shortness of breath using harmonized baseline data from two very large European cohorts, Lifelines (2006-2013) and UK Biobank (2006-2010). Our aim was also to determine whether the relationship between air pollution and respiratory symptom prevalence differed between individuals with different characteristics.', 'Cross-sectional analyses explored associations between prevalence of self-reported wheeze and shortness of breath and annual mean particulate matter with aerodynamic diameter &lt;2.5\u03bcm, 2.5-10\u03bcm, and &lt;10\u03bcm (PM2.5, PMcoarse, and PM10, respectively) and nitrogen dioxide (NO2) concentrations at place of residence using logistic regression. Subgroup analyses and tests for interaction were performed for age, sex, smoking status, household income, obesity status, and asthma status.', 'All PM exposures were associated with respiratory symptoms based on single-pollutant models, with the largest associations seen for PM2.5 with prevalence of wheezing {odds ratio (OR)=1.16 per 5\u03bcg/m\u00b3 [95% confidence interval (CI): 1.11, 1.21]} and shortness of breath [OR=1.61 per 5\u03bcg/m\u00b3 (95% CI: 1.45, 1.78)]. The association between shortness of breath and a 5-\u03bcg/m\u00b3 increment in PM2.5 was significantly higher for individuals from lower-[OR=1.73 (95% CI: 1.52, 1.97)] versus higher-income households [OR=1.31 (95% CI: 1.11, 1.55); p-interaction=0.005), whereas the association between PM2.5 and wheeze was limited to lower-income participants [OR=1.30 (95% CI: 1.22, 1.38) vs. OR=1.02; (95% CI: 0.96, 1.08); p-interaction&lt;0.001]. Exposure to NO2 also showed positive associations with wheeze and shortness of breath.', 'Exposure to PM and NO2 air pollution was associated with the prevalence of wheeze and shortness of breath in this large study, with stronger associations between PM2.5 and both outcomes among lower- versus higher-income participants. https://doi.org/10.1289/EHP1353.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Air Pollutants", "major": "N"}, {"term": "Air Pollution", "major": "N"}, {"term": "Cross-Sectional Studies", "major": "N"}, {"term": "Dyspnea", "major": "N"}, {"term": "Environmental Exposure", "major": "N"}, {"term": "Europe", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Logistic Models", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Nitrogen Dioxide", "major": "N"}, {"term": "Prevalence", "major": "N"}, {"term": "Respiratory Sounds", "major": "Y"}, {"term": "Risk Factors", "major": "N"}, {"term": "Vehicle Emissions", "major": "N"}, {"term": "Young Adult", "major": "N"}], "other": []}, "full_author_list": ["Doiron D", "de Hoogh K", "Probst-Hensch N", "Mbatchou S", "Eeftens M", "Cai Y", "Schindler C", "Fortier I", "Hodgson S", "Gaye A", "Stolk R", "Hansell A"], "journal": {"journal_name": "Environ Health Perspect", "volume": "125", "issue": "9"}}, {"IDs": {"DOI": "10.1186/1472-6823-14-9", "PMID": "24484869"}, "title": "The prevalence of metabolic syndrome and metabolically healthy obesity in Europe: a collaborative analysis of ten large cohort studies.", "abstract": "['Not all obese subjects have an adverse metabolic profile predisposing them to developing type 2 diabetes or cardiovascular disease. The BioSHaRE-EU Healthy Obese Project aims to gain insights into the consequences of (healthy) obesity using data on risk factors and phenotypes across several large-scale cohort studies. Aim of this study was to describe the prevalence of obesity, metabolic syndrome (MetS) and metabolically healthy obesity (MHO) in ten participating studies.', 'Ten different cohorts in seven countries were combined, using data transformed into a harmonized format. All participants were of European origin, with age 18-80\\xa0years. They had participated in a clinical examination for anthropometric and blood pressure measurements. Blood samples had been drawn for analysis of lipids and glucose. Presence of MetS was assessed in those with obesity (BMI\\u2009\u2265\\u200930\\xa0kg/m2) based on the 2001 NCEP ATP III criteria, as well as an adapted set of less strict criteria. MHO was defined as obesity, having none of the MetS components, and no previous diagnosis of cardiovascular disease.', 'Data for 163,517 individuals were available; 17% were obese (11,465 men and 16,612 women). The prevalence of obesity varied from 11.6% in the Italian CHRIS cohort to 26.3% in the German KORA cohort. The age-standardized percentage of obese subjects with MetS ranged in women from 24% in CHRIS to 65% in the Finnish Health2000 cohort, and in men from 43% in CHRIS to 78% in the Finnish DILGOM cohort, with elevated blood pressure the most frequently occurring factor contributing to the prevalence of the metabolic syndrome. The age-standardized prevalence of MHO varied in women from 7% in Health2000 to 28% in NCDS, and in men from 2% in DILGOM to 19% in CHRIS. MHO was more prevalent in women than in men, and decreased with age in both sexes.', 'Through a rigorous harmonization process, the BioSHaRE-EU consortium was able to compare key characteristics defining the metabolically healthy obese phenotype across ten cohort studies. There is considerable variability in the prevalence of healthy obesity across the different European populations studied, even when unified criteria were used to classify this phenotype.']", "keywords": {"mesh": [], "other": []}, "full_author_list": ["van Vliet-Ostaptchouk J", "Nuotio M", "Slagter S", "Doiron D", "Fischer K", "Foco L", "Gaye A", "G\u00f6gele M", "Heier M", "Hiekkalinna T", "Joensuu A", "Newby C", "Pang C", "Partinen E", "Reischl E", "Schwienbacher C", "Tammesoo M", "Swertz M", "Burton P", "Ferretti V", "Fortier I", "Giepmans L", "Harris J", "Hillege H", "Holmen J", "Jula A", "Kootstra-Ros J", "Kval\u00f8y K", "Holmen T", "M\u00e4nnist\u00f6 S", "Metspalu A", "Midthjell K", "Murtagh M", "Peters A", "Pramstaller P", "Saaristo T", "Salomaa V", "Stolk R", "Uusitupa M", "van der Harst P", "van der Klauw M", "Waldenberger M", "Perola M", "Wolffenbuttel B"], "journal": {"journal_name": "BMC Endocr Disord", "volume": "14", "issue": ""}}, {"IDs": {"DOI": "10.1371/journal.pcbi.1008880", "PMID": "33784300"}, "title": "Orchestrating privacy-protected big data analyses of data from different resources with R and DataSHIELD.", "abstract": "['Combined analysis of multiple, large datasets is a common objective in the health- and biosciences. Existing methods tend to require researchers to physically bring data together in one place or follow an analysis plan and share results. Developed over the last 10 years, the DataSHIELD platform is a collection of R packages that reduce the challenges of these methods. These include ethico-legal constraints which limit researchers\\' ability to physically bring data together and the analytical inflexibility associated with conventional approaches to sharing results. The key feature of DataSHIELD is that data from research studies stay on a server at each of the institutions that are responsible for the data. Each institution has control over who can access their data. The platform allows an analyst to pass commands to each server and the analyst receives results that do not disclose the individual-level data of any study participants. DataSHIELD uses Opal which is a data integration system used by epidemiological studies and developed by the OBiBa open source project in the domain of bioinformatics. However, until now the analysis of big data with DataSHIELD has been limited by the storage formats available in Opal and the analysis capabilities available in the DataSHIELD R packages. We present a new architecture (\"resources\") for DataSHIELD and Opal to allow large, complex datasets to be used at their original location, in their original format and with external computing facilities. We provide some real big data analysis examples in genomics and geospatial projects. For genomic data analyses, we also illustrate how to extend the resources concept to address specific big data infrastructures such as GA4GH or EGA, and make use of shell commands. Our new infrastructure will help researchers to perform data analyses in a privacy-protected way from existing data sharing initiatives or projects. To help researchers use this framework, we describe selected packages and present an online book (https://isglobal-brge.github.io/resource_bookdown).']", "keywords": {"mesh": [{"term": "Big Data", "major": "Y"}, {"term": "Computer Security", "major": "Y"}, {"term": "Databases, Factual", "major": "N"}, {"term": "Genomics", "major": "N"}, {"term": "Geographic Information Systems", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Software", "major": "Y"}], "other": []}, "full_author_list": ["Marcon Y", "Bishop T", "Avraam D", "Escriba-Montagut X", "Ryser-Welch P", "Wheater S", "Burton P", "Gonz\u00e1lez J"], "journal": {"journal_name": "PLoS Comput Biol", "volume": "17", "issue": "3"}}, {"IDs": {"DOI": "10.1093/bioinformatics/btv279", "PMID": "26112289"}, "title": "Data Safe Havens in health research and healthcare.", "abstract": "[\"The data that put the 'evidence' into 'evidence-based medicine' are central to developments in public health, primary and hospital care. A fundamental challenge is to site such data in repositories that can easily be accessed under appropriate technical and governance controls which are effectively audited and are viewed as trustworthy by diverse stakeholders. This demands socio-technical solutions that may easily become enmeshed in protracted debate and controversy as they encounter the norms, values, expectations and concerns of diverse stakeholders. In this context, the development of what are called 'Data Safe Havens' has been crucial. Unfortunately, the origins and evolution of the term have led to a range of different definitions being assumed by different groups. There is, however, an intuitively meaningful interpretation that is often assumed by those who have not previously encountered the term: a repository in which useful but potentially sensitive data may be kept securely under governance and informatics systems that are fit-for-purpose and appropriately tailored to the nature of the data being maintained, and may be accessed and utilized by legitimate users undertaking work and research contributing to biomedicine, health and/or to ongoing development of healthcare systems.\", \"This review explores a fundamental question: 'what are the specific criteria that ought reasonably to be met by a data repository if it is to be seen as consistent with this interpretation and viewed as worthy of being accorded the status of 'Data Safe Haven' by key stakeholders'? We propose 12 such criteria.\", 'paul.burton@bristol.ac.uk.']", "keywords": {"mesh": [{"term": "Access to Information", "major": "Y"}, {"term": "Biomedical Research", "major": "Y"}, {"term": "Confidentiality", "major": "Y"}, {"term": "Delivery of Health Care", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Research", "major": "N"}], "other": []}, "full_author_list": ["Burton P", "Murtagh M", "Boyd A", "Williams J", "Dove E", "Wallace S", "Tass\u00e9 A", "Little J", "Chisholm R", "Gaye A", "Hveem K", "Brookes A", "Goodwin P", "Fistein J", "Bobrow M", "Knoppers B"], "journal": {"journal_name": "Bioinformatics", "volume": "31", "issue": "20"}}, {"IDs": {"DOI": "10.1186/s12888-021-03234-x", "PMID": "33952234"}, "title": "Bidimensional structure and measurement equivalence of the Patient Health Questionnaire-9: sex-sensitive assessment of depressive symptoms in three representative German cohort studies.", "abstract": "['The Patient Health Questionnaire-9 (PHQ-9) has been proposed as a reliable and valid screening instrument for depressive symptoms with one latent factor. However, studies explicitly testing alternative model structures found support for a two-dimensional structure reflecting a somatic and a cognitive-affective dimension. We investigated the bidimensional structure of the PHQ-9, with a somatic (sleeping problems, fatigability, appetitive problems, and psychomotor retardation) and a cognitive-affective dimension (lack of interest, depressed mood, negative feelings about self, concentration problems, and suicidal ideation), and tested for sex- and regional-differences.', 'We have included data from the GEnder-Sensitive Analyses of mental health trajectories and implications for prevention: A multi-cohort consortium (GESA). Privacy-preserving analyses to provide information on the overall population and cohort-specific information and analyses of variance to compare depressive, somatic and cognitive-affective symptoms between sexes and cohorts were executed in DataSHIELD. In order to determine the dimensionality and measurement invariance of the PHQ-9 we tested three models (1 factor, 2 correlated factors, and bifactor) via confirmatory analyses and performed multi-group confirmatory factor analysis.', 'Differences between sex and cohorts exist for PHQ-9 and for both of its dimensions. Women reported depressive symptoms in general as well as somatic and cognitive-affective symptoms more frequently. For all tested models an acceptable to excellent fit was found, consistently indicating a better model fit for the two-factor and bifactor model. Scalar measurement invariance was established between women and men, the three cohorts, and their interaction.', 'The two facets of depression should be taken into account when using PHQ-9, while data also render support to a general factor. Somatic and cognitive-affective symptoms assessed by the PHQ-9 can be considered equivalent across women and men and between different German populations from different regions.']", "keywords": {"mesh": [{"term": "Cohort Studies", "major": "N"}, {"term": "Depression", "major": "Y"}, {"term": "Factor Analysis, Statistical", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Patient Health Questionnaire", "major": "Y"}, {"term": "Psychometrics", "major": "N"}, {"term": "Surveys and Questionnaires", "major": "N"}], "other": ["Cognitive-affective dimension", "Depression", "Regional differences", "Sex-differences", "Somatic dimension"]}, "full_author_list": ["Tibubos A", "Otten D", "Z\u00f6ller D", "Binder H", "Wild P", "Fleischer T", "Johar H", "Atasoy S", "Schulze L", "Ladwig K", "Schomerus G", "Linkohr B", "Grabe H", "Kruse J", "Schmidt C", "M\u00fcnzel T", "K\u00f6nig J", "Br\u00e4hler E", "Beutel M"], "journal": {"journal_name": "BMC Psychiatry", "volume": "21", "issue": "1"}}, {"IDs": {"DOI": "10.1186/1742-7622-10-12", "PMID": "24257327"}, "title": "Data harmonization and federated analysis of population-based studies: the BioSHaRE project.", "abstract": "['Individual-level data pooling of large population-based studies across research centres in international research projects faces many hurdles. The BioSHaRE (Biobank Standardisation and Harmonisation for Research Excellence in the European Union) project aims to address these issues by building a collaborative group of investigators and developing tools for data harmonization, database integration and federated data analyses.', \"Eight population-based studies in six European countries were recruited to participate in the BioSHaRE project. Through workshops, teleconferences and electronic communications, participating investigators identified a set of 96 variables targeted for harmonization to answer research questions of interest. Using each study's questionnaires, standard operating procedures, and data dictionaries, harmonization potential was assessed. Whenever harmonization was deemed possible, processing algorithms were developed and implemented in an open-source software infrastructure to transform study-specific data into the target (i.e. harmonized) format. Harmonized datasets located on server in each research centres across Europe were interconnected through a federated database system to perform statistical analysis.\", 'Retrospective harmonization led to the generation of common format variables for 73% of matches considered (96 targeted variables across 8 studies). Authenticated investigators can now perform complex statistical analyses of harmonized datasets stored on distributed servers without actually sharing individual-level data using the DataSHIELD method.', 'New Internet-based networking technologies and database management systems are providing the means to support collaborative, multi-center research in an efficient and secure manner. The results from this pilot project show that, given a strong collaborative relationship between participating studies, it is possible to seamlessly co-analyse internationally harmonized research databases while allowing each study to retain full control over individual-level data. We encourage additional collaborative research networks in epidemiology, public health, and the social sciences to make use of the open source tools presented herein.']", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Doiron D", "Burton P", "Marcon Y", "Gaye A", "Wolffenbuttel B", "Perola M", "Stolk R", "Foco L", "Minelli C", "Waldenberger M", "Holle R", "Kval\u00f8y K", "Hillege H", "Tass\u00e9 A", "Ferretti V", "Fortier I"], "journal": {"journal_name": "Emerg Themes Epidemiol", "volume": "10", "issue": "1"}}, {"IDs": {"DOI": "10.1007/s10654-020-00662-z", "PMID": "32705500"}, "title": "The LifeCycle Project-EU Child Cohort Network: a federated analysis infrastructure and harmonized data of more than 250,000 children and parents.", "abstract": "['Early life is an important window of opportunity to improve health across the full lifecycle. An accumulating body of evidence suggests that exposure to adverse stressors during early life leads to developmental adaptations, which subsequently affect disease risk in later life. Also, geographical, socio-economic, and ethnic differences are related to health inequalities from early life onwards. To address these important public health challenges, many European pregnancy and childhood cohorts have been established over the last 30\\xa0years. The enormous wealth of data of these cohorts has led to important new biological insights and important impact for health from early life onwards. The impact of these cohorts and their data could be further increased by combining data from different cohorts. Combining data will lead to the possibility of identifying smaller effect estimates, and the opportunity to better identify risk groups and risk factors leading to disease across the lifecycle across countries. Also, it enables research on better causal understanding and modelling of life course health trajectories. The EU Child Cohort Network, established by the Horizon2020-funded LifeCycle Project, brings together nineteen pregnancy and childhood cohorts, together including more than 250,000 children and their parents. A large set of variables has been harmonised and standardized across these cohorts. The harmonized data are kept within each institution and can be accessed by external researchers through a shared federated data analysis platform using the R-based platform DataSHIELD, which takes relevant national and international data regulations into account. The EU Child Cohort Network has an open character. All protocols for data harmonization and setting up the data analysis platform are available online. The EU Child Cohort Network creates great opportunities for researchers to use data from different cohorts, during and beyond the LifeCycle Project duration. It also provides a novel model for collaborative research in large research infrastructures with individual-level data. The LifeCycle Project will translate results from research using the EU Child Cohort Network into recommendations for targeted prevention strategies to improve health trajectories for current and future generations by optimizing their earliest phases of life.']", "keywords": {"mesh": [{"term": "Adolescent", "major": "N"}, {"term": "Child", "major": "N"}, {"term": "Child, Preschool", "major": "N"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Databases, Factual", "major": "N"}, {"term": "Environmental Exposure", "major": "Y"}, {"term": "Environmental Health", "major": "N"}, {"term": "European Union", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Infant", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Noncommunicable Diseases", "major": "Y"}, {"term": "Parents", "major": "N"}, {"term": "Pregnancy", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Socioeconomic Factors", "major": "N"}], "other": ["Birth cohorts", "Consortium", "Exposome", "Life course", "Non-communicable diseases"]}, "full_author_list": ["Jaddoe V", "Felix J", "Andersen A", "Charles M", "Chatzi L", "Corpeleijn E", "Donner N", "Elhakeem A", "Eriksson J", "Foong R", "Grote V", "Haakma S", "Hanson M", "Harris J", "Heude B", "Huang R", "Inskip H", "J\u00e4rvelin M", "Koletzko B", "Lawlor D", "Lindeboom M", "McEachan R", "Mikkola T", "Nader J", "de Moira A", "Pizzi C", "Richiardi L", "Sebert S", "Schwalber A", "Sunyer J", "Swertz M", "Vafeiadi M", "Vrijheid M", "Wright J", "Duijts L"], "journal": {"journal_name": "Eur J Epidemiol", "volume": "35", "issue": "7"}}, {"IDs": {"DOI": "10.48550/ARXIV.1803.00422", "PMID": ""}, "title": "Distributed Multivariate Regression Modeling For Selecting Biomarkers Under Data Protection Constraints", "abstract": "The discovery of clinical biomarkers requires large patient cohorts and is aided by a pooled data approach across institutions. In many countries, data protection constraints, especially in the clinical environment, forbid the exchange of individual-level data between different research institutes, impeding the conduct of a joint analyses. To circumvent this problem, only non-disclosive aggregated data is exchanged, which is often done manually and requires explicit permission before transfer, i.e., the number of data calls and the amount of data should be limited. This does not allow for more complex tasks such as variable selection, as only simple aggregated summary statistics are typically transferred. Other methods have been proposed that require more complex aggregated data or use input data perturbation, but these methods can either not deal with a high number of biomarkers or lose information. Here, we propose a multivariable regression approach for identifying biomarkers by automatic variable selection based on aggregated data in iterative calls, which can be implemented under data protection constraints. The approach can be used to jointly analyze data distributed across several locations. To minimize the amount of transferred data and the number of calls, we also provide a heuristic variant of the approach. When performing global data standardization, the proposed method yields the same results as pooled individual-level data analysis. In a simulation study, the information loss introduced by local standardization is seen to be minimal. In a typical scenario, the heuristic decreases the number of data calls from more than 10 to 3, rendering manual data releases feasible. To make our approach widely available for application, we provide an implementation of the heuristic version incorporated in the DataSHIELD framework.\\", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Z\u00f6ller D", "Binder H"], "journal": {"journal_name": "", "volume": "", "issue": ""}}, {"IDs": {"DOI": "10.3390/life14020262", "PMID": "38398771"}, "title": "A Federated Database for Obesity Research: An IMI-SOPHIA Study.", "abstract": "['Obesity is considered by many as a lifestyle choice rather than a chronic progressive disease. The Innovative Medicines Initiative (IMI) SOPHIA (Stratification of Obesity Phenotypes to Optimize Future Obesity Therapy) project is part of a momentum shift aiming to provide better tools for the stratification of people with obesity according to disease risk and treatment response. One of the challenges to achieving these goals is that many clinical cohorts are siloed, limiting the potential of combined data for biomarker discovery. In SOPHIA, we have addressed this challenge by setting up a federated database building on open-source DataSHIELD technology. The database currently federates 16 cohorts that are accessible via a central gateway. The database is multi-modal, including research studies, clinical trials, and routine health data, and is accessed using the R statistical programming environment where statistical and machine learning analyses can be performed at a distance without any disclosure of patient-level data. We demonstrate the use of the database by providing a proof-of-concept analysis, performing a federated linear model of BMI and systolic blood pressure, pooling all data from 16 studies virtually without any analyst seeing individual patient-level data. This analysis provided similar point estimates compared to a meta-analysis of the 16 individual studies. Our approach provides a benchmark for reproducible, safe federated analyses across multiple study types provided by multiple stakeholders.']", "keywords": {"mesh": [], "other": ["bioinformatics", "federated database system", "obesity", "remote statistical analysis", "risk prediction"]}, "full_author_list": ["Delfin C", "Dragan I", "Kuznetsov D", "Tajes J", "Smit F", "Coral D", "Farzaneh A", "Haugg A", "Hungele A", "Niknejad A", "Hall C", "Jacobs D", "Marek D", "Fraser D", "Thuillier D", "Ahmadizar F", "Mehl F", "Pattou F", "Burdet F", "Hawkes G", "Arts I", "Blanch J", "Van Soest J", "Fern\u00e1ndez-Real J", "Boehl J", "Fink K", "van Greevenbroek M", "Kavousi M", "Minten M", "Prinz N", "Ipsen N", "Franks P", "Ramos R", "Holl R", "Horban S", "Duarte-Salles T", "Tran V", "Raverdy V", "Leal Y", "Lenart A", "Pearson E", "Spars\u00f8 T", "Giordano G", "Ioannidis V", "Soh K", "Frayling T", "Le Roux C", "Ibberson M"], "journal": {"journal_name": "Life (Basel)", "volume": "14", "issue": "2"}}, {"IDs": {"DOI": "10.12688/f1000research.8736.1", "PMID": ""}, "title": "The Biomedical Research Infrastructure Software as a Service Kit (BRISSKit): technical description", "abstract": "With biomedical research becoming ever more computationally intensive, the challenge is to find sophisticated software tools that can keep pace with new requirements, while still being easy to use and secure. We describe a technical implementation of an infrastructure to manage the full research ecosystem from participant management, to data and sample collection, and finally to data storage, interrogation and analysis. This infrastructure, known as the Biomedical Research Infrastructure Software as a Service Kit (BRISSKit http://www.brisskit.le.ac.uk), is built on open source solutions throughout, and demonstrates that it is possible for a biomedical research platform to be supplied as a service.", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Butters O", "Issa S", "Lusted J", "Newbury M", "Parsloe R", "Holden N", "Free R", "Beck T", "Wilson R", "Burton P", "Tedds J"], "journal": {"journal_name": "F1000Research", "volume": "5", "issue": ""}}, {"IDs": {"DOI": "10.1007/s00394-022-02909-9", "PMID": "35641800"}, "title": "Associations between exploratory dietary patterns and incident type 2 diabetes: a federated meta-analysis of individual participant data from 25 cohort studies.", "abstract": "['In several studies, exploratory dietary patterns (DP), derived by principal component analysis, were inversely or positively associated with incident type 2 diabetes (T2D). However, findings remained study-specific, inconsistent and rarely replicated. This study aimed to investigate the associations between DPs and T2D in multiple cohorts across the world.', 'This federated meta-analysis of individual participant data was based on 25 prospective cohort studies from 5 continents including a total of 390,664 participants with a follow-up for T2D (3.8-25.0\\xa0years). After data harmonization across cohorts we evaluated 15 previously identified T2D-related DPs for association with incident T2D estimating pooled incidence rate ratios (IRR) and confidence intervals (CI) by Piecewise Poisson regression and random-effects meta-analysis.', '29,386 participants developed T2D during follow-up. Five DPs, characterized by higher intake of red meat, processed meat, French fries and refined grains, were associated with higher incidence of T2D. The strongest association was observed for a DP comprising these food groups besides others (IRR&lt;sub&gt;pooled&lt;/sub&gt; per 1 SD\\u2009=\\u20091.104, 95% CI 1.059-1.151). Although heterogeneity was present (I&lt;sup&gt;2&lt;/sup&gt;\\u2009=\\u200985%), IRR exceeded 1 in 18 of the 20 meta-analyzed studies. Original DPs associated with lower T2D risk were not confirmed. Instead, a healthy DP (HDP1) was associated with higher T2D risk (IRR&lt;sub&gt;pooled&lt;/sub&gt; per 1 SD\\u2009=\\u20091.057, 95% CI 1.027-1.088).', 'Our findings from various cohorts revealed positive associations for several DPs, characterized by higher intake of red meat, processed meat, French fries and refined grains, adding to the evidence-base that links DPs to higher T2D risk. However, no inverse DP-T2D associations were confirmed.']", "keywords": {"mesh": [{"term": "Cohort Studies", "major": "N"}, {"term": "Diabetes Mellitus, Type 2", "major": "Y"}, {"term": "Diet", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Incidence", "major": "N"}, {"term": "Prospective Studies", "major": "N"}, {"term": "Risk Factors", "major": "N"}], "other": ["Dietary patterns", "Exploratory", "Federated meta-analysis", "Type 2 diabetes mellitus"]}, "full_author_list": ["Jannasch F", "Dietrich S", "Bishop T", "Pearce M", "Fanidi A", "O'Donoghue G", "O'Gorman D", "Marques-Vidal P", "Vollenweider P", "Bes-Rastrollo M", "Byberg L", "Wolk A", "Hashemian M", "Malekzadeh R", "Poustchi H", "Luft V", "de Matos S", "Kim J", "Kim M", "Kim Y", "Stern D", "Lajous M", "Magliano D", "Shaw J", "Akbaraly T", "Kivimaki M", "Maskarinec G", "Le Marchand L", "Mart\u00ednez-Gonz\u00e1lez M", "Soedamah-Muthu S", "Wareham N", "Forouhi N", "Schulze M"], "journal": {"journal_name": "Eur J Nutr", "volume": "61", "issue": "7"}}, {"IDs": {"DOI": "10.1093/jn/nxaa447", "PMID": "33693815"}, "title": "Associations of Total Legume, Pulse, and Soy Consumption with Incident Type 2 Diabetes: Federated Meta-Analysis of 27 Studies from Diverse World Regions.", "abstract": "['The consumption of legumes is promoted as part of a healthy diet in many countries but associations of total and types of legume consumption with type 2 diabetes (T2D) are not well established. Analyses across diverse populations are lacking despite the availability of unpublished legume consumption data in prospective cohort studies.', 'To examine the prospective associations of total and types of legume intake with the risk of incident T2D.', 'Meta-analyses of associations between total legume, pulse, and soy consumption and T2D were conducted using a federated approach without physical data-pooling. Prospective cohorts were included if legume exposure and T2D outcome data were available and the cohort investigators agreed to participate. We estimated incidence rate ratios (IRRs) and CIs of associations using individual participant data including \u226442,473 incident cases among 807,785 adults without diabetes in 27 cohorts across the Americas, Eastern Mediterranean, Europe, and Western Pacific. Random-effects meta-analysis was used to combine effect estimates and estimate heterogeneity.', 'Median total legume intake ranged from 0-140\\xa0g/d across cohorts. We observed a weak positive association between total legume consumption and T2D (IRR\\xa0=\\xa01.02, 95% CI: 1.01 to 1.04) per 20\\xa0g/d higher intake, with moderately high heterogeneity (I2\\xa0=\\xa074%). Analysis by region showed no evidence of associations in the Americas, Eastern Mediterranean, and Western Pacific. The positive association in Europe (IRR\\xa0=\\xa01.05, 95% CI: 1.01 to 1.10, I2\\xa0=\\xa082%) was mainly driven by studies from Germany, UK, and Sweden. No evidence of associations was observed for the consumption of pulses or soy.', 'These findings suggest no evidence of an association of legume intakes with T2D in several world regions. The positive association observed in some European studies warrants further investigation relating to overall dietary contexts in which legumes are consumed, including accompanying foods which may be positively associated with T2D.']", "keywords": {"mesh": [{"term": "Cohort Studies", "major": "N"}, {"term": "Diabetes Mellitus, Type 2", "major": "Y"}, {"term": "Diet", "major": "Y"}, {"term": "Fabaceae", "major": "Y"}, {"term": "Global Health", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Incidence", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Soybean Proteins", "major": "Y"}], "other": ["bean", "chickpea", "diabetes", "legumes", "lentil", "pea", "peanut", "pulse", "soy"]}, "full_author_list": ["Pearce M", "Fanidi A", "Bishop T", "Sharp S", "Imamura F", "Dietrich S", "Akbaraly T", "Bes-Rastrollo M", "Beulens J", "Byberg L", "Canhada S", "Molina M", "Chen Z", "Cortes-Valencia A", "Du H", "Duncan B", "H\u00e4rk\u00e4nen T", "Hashemian M", "Kim J", "Kim M", "Kim Y", "Knekt P", "Kromhout D", "Lassale C", "Ridaura R", "Magliano D", "Malekzadeh R", "Marques-Vidal P", "Mart\u00ednez-Gonz\u00e1lez M", "O'Donoghue G", "O'Gorman D", "Shaw J", "Soedamah-Muthu S", "Stern D", "Wolk A", "Woo H", "Wareham N", "Forouhi N"], "journal": {"journal_name": "J Nutr", "volume": "151", "issue": "5"}}, {"IDs": {"DOI": "10.1159/000360255", "PMID": "24685519"}, "title": "Protecting personal data in epidemiological research: DataSHIELD and UK law.", "abstract": "['Data from individual collections, such as biobanks and cohort studies, are now being shared in order to create combined datasets which can be queried to ask complex scientific questions. But this sharing must be done with due regard for data protection principles. DataSHIELD is a new technology that queries nonaggregated, individual-level data in situ but returns query data in an anonymous format. This raises questions of the ability of DataSHIELD to adequately protect participant confidentiality.', 'An ethico-legal analysis was conducted that examined each step of the DataSHIELD process from the perspective of UK case law, regulations, and guidance.', 'DataSHIELD reaches agreed UK standards of protection for the sharing of biomedical data. All direct processing of personal data is conducted within the protected environment of the contributing study; participating studies have scientific, ethics, and data access approvals in place prior to the analysis; studies are clear that their consents conform with this use of data, and participants are informed that anonymisation for further disclosure will take place.', \"DataSHIELD can provide a flexible means of interrogating data while protecting the participants' confidentiality in accordance with applicable legislation and guidance.\"]", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "N"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Computer Security", "major": "N"}, {"term": "Confidentiality", "major": "N"}, {"term": "Databases, Factual", "major": "Y"}, {"term": "Epidemiologic Studies", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Information Dissemination", "major": "N"}, {"term": "Informed Consent", "major": "N"}, {"term": "Research Design", "major": "N"}, {"term": "United Kingdom", "major": "N"}], "other": []}, "full_author_list": ["Wallace S", "Gaye A", "Shoush O", "Burton P"], "journal": {"journal_name": "Public Health Genomics", "volume": "17", "issue": "3"}}, {"IDs": {"DOI": "10.1186/s12874-021-01237-6", "PMID": "33812380"}, "title": "Deep generative models in DataSHIELD.", "abstract": "['The best way to calculate statistics from medical data is to use the data of individual patients. In some settings, this data is difficult to obtain due to privacy restrictions. In Germany, for example, it is not possible to pool routine data from different hospitals for research purposes without the consent of the patients.', 'The DataSHIELD software provides an infrastructure and a set of statistical methods for joint, privacy-preserving analyses of distributed data. The contained algorithms are reformulated to work with aggregated data from the participating sites instead of the individual data. If a desired algorithm is not implemented in DataSHIELD or cannot be reformulated in such a way, using artificial data is an alternative. Generating artificial data is possible using so-called generative models, which are able to capture the distribution of given data. Here, we employ deep Boltzmann machines (DBMs) as generative models. For the implementation, we use the package \"BoltzmannMachines\" from the Julia programming language and wrap it for use with DataSHIELD, which is based on R.', 'We present a methodology together with a software implementation that builds on DataSHIELD to create artificial data that preserve complex patterns from distributed individual patient data. Such data sets of artificial patients, which are not linked to real patients, can then be used for joint analyses. As an exemplary application, we conduct a distributed analysis with DBMs on a synthetic data set, which simulates genetic variant data. Patterns from the original data can be recovered in the artificial data using hierarchical clustering of the virtual patients, demonstrating the feasibility of the approach. Additionally, we compare DBMs, variational autoencoders, generative adversarial networks, and multivariate imputation as generative approaches by assessing the utility and disclosure of synthetic data generated from real genetic variant data in a distributed setting with data of a small sample size.', 'Our implementation adds to DataSHIELD the ability to generate artificial data that can be used for various analyses, e.g., for pattern recognition with deep learning. This also demonstrates more generally how DataSHIELD can be flexibly extended with advanced algorithms from languages other than R.']", "keywords": {"mesh": [{"term": "Algorithms", "major": "Y"}, {"term": "Disclosure", "major": "N"}, {"term": "Germany", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Software", "major": "Y"}], "other": ["Biomedical research/methods", "Deep learning", "Distributed system", "Privacy/statistics and numerical data"]}, "full_author_list": ["Lenz S", "Hess M", "Binder H"], "journal": {"journal_name": "BMC Med Res Methodol", "volume": "21", "issue": "1"}}, {"IDs": {"DOI": "10.1136/bmjopen-2019-034220", "PMID": "32102819"}, "title": "A multi-cohort consortium for GEnder-Sensitive Analyses of mental health trajectories and implications for prevention (GESA) in the general population in Germany.", "abstract": "['Mental health is marked by gender differences. We formed a multi-cohort consortium to perform GEnder-Sensitive Analyses of mental health trajectories and study their implications for prevention (GESA). GESA aims at (1) identifying gender differences regarding symptoms and trajectories of mental health over the lifespan; (2) determining gender differences regarding the prevalence, impact of risk and protective factors; and (3) determining effects of mental health on primary and secondary outcomes (eg, quality of life, healthcare behaviour and utilisation).', 'We plan to perform secondary analyses on three major, ongoing, population-based, longitudinal cohorts (Gutenberg Health-Study (GHS), Study of Health in Pomerania (SHIP), Cooperative Health Research in the Augsburg Region (KORA)) with data on mental and somatic symptoms, medical assessments and diagnoses in north-east, middle and southern Germany (n&gt;40\\u2009000). Meta-analytic techniques (using DataSHIELD framework) will be used to combine aggregated data from these cohorts. This process will inform about heterogeneity of effects. Longitudinal regression models will estimate sex-specific trajectories and effects of risk and protective factors and secondary outcomes.', 'The cohorts were approved by the ethics committees of the Statutory Physician Board of Rhineland-Palatinate (837.020.07; GHS), the University of Greifswald (BB 39/08; SHIP) and the Bavarian Chamber of Physicians (06068; KORA). Together with stakeholders in medical care and medical training, findings will be translated and disseminated into gender-sensitive health promotion and prevention.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Aged", "major": "N"}, {"term": "Clinical Protocols", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Germany", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Longitudinal Studies", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Mental Disorders", "major": "N"}, {"term": "Mental Health", "major": "N"}, {"term": "Middle Aged", "major": "N"}, {"term": "Prevalence", "major": "N"}, {"term": "Regression Analysis", "major": "N"}, {"term": "Risk Factors", "major": "N"}, {"term": "Sex Factors", "major": "N"}], "other": ["assessment", "common mental disorders", "gender", "prospective", "sex"]}, "full_author_list": ["Burghardt J", "Tibubos A", "Otten D", "Br\u00e4hler E", "Binder H", "Grabe H", "Kruse J", "Ladwig K", "Schomerus G", "Wild P", "Beutel M"], "journal": {"journal_name": "BMJ Open", "volume": "10", "issue": "2"}}, {"IDs": {"DOI": "10.1186/s13104-023-06372-5", "PMID": "37280717"}, "title": "dsSurvival 2.0: privacy enhancing survival curves for survival models in the federated DataSHIELD analysis system.", "abstract": "['Survival models are used extensively in biomedical sciences, where they allow the investigation of the effect of exposures on health outcomes. It is desirable to use diverse data sets in survival analyses, because this offers increased statistical power and generalisability of results. However, there are often challenges with bringing data together in one location or following an analysis plan and sharing results. DataSHIELD is an analysis platform that helps users to overcome these ethical, governance and process difficulties. It allows users to analyse data remotely, using functions that are built to restrict access to the detailed data items (federated analysis). Previous works have provided survival modelling functionality in DataSHIELD (dsSurvival package), but there is a requirement to provide functions that offer privacy enhancing survival curves that retain useful information.', 'We introduce an enhanced version of the dsSurvival package which offers privacy enhancing survival curves for DataSHIELD. Different methods for enhancing privacy were evaluated for their effectiveness in enhancing privacy while maintaining utility. We demonstrated how our selected method could enhance privacy in different scenarios using real survival data. The details of how DataSHIELD can be used to generate survival curves can be found in the associated tutorial.']", "keywords": {"mesh": [{"term": "Confidentiality", "major": "N"}, {"term": "Privacy", "major": "Y"}, {"term": "Survival Analysis", "major": "Y"}, {"term": "Data Science", "major": "Y"}, {"term": "Data Anonymization", "major": "N"}, {"term": "Data Analysis", "major": "N"}, {"term": "Models, Statistical", "major": "Y"}, {"term": "Ethics, Research", "major": "N"}], "other": ["DataSHIELD", "Privacy preserving federated analysis", "Survival curves"]}, "full_author_list": ["Banerjee S", "Bishop T"], "journal": {"journal_name": "BMC Res Notes", "volume": "16", "issue": "1"}}, {"IDs": {"DOI": "10.1038/s41598-021-93242-4", "PMID": "34234182"}, "title": "The relation between childhood adversity and adult obesity in a population-based study in women and men.", "abstract": "['Childhood maltreatment has been shown to relate to adult obesity. In this epidemiological study, we investigate the association between childhood maltreatment and waist-to-height-ratio (WHtR) in a sample of the German adult population, comprising of N\\u2009=\\u20092936 participants. WHtR, an indicator for risk of obesity, was the primary outcome. Childhood maltreatment was assessed by the Childhood Trauma Screener (CTS), which assesses emotional and physical neglect, abuse as well as sexual abuse. Cohort-data were harmonized and analyzed within DataSHIELD. We used multivariable regression models to estimate the association of childhood maltreatment and WHtR at different levels of adjustments for potential confounders. Overall childhood maltreatment was associated with a higher WHtR in both sexes (women: p\\u2009=\\u20090.004, men: p\\u2009&lt;\\u20090.001); associations were no longer significant in women after adding socioeconomic variables, but remained significant in men (p\\u2009=\\u20090.013). Additionally, we were able to identify sex specific patterns for childhood maltreatment predicting the WHtR. Emotional neglect and abuse had stronger impacts on the WHtR in women than in men, whereas physical neglect and abuse had stronger impacts in men. To our knowledge, this is the first comprehensive population-based study testing various types of childhood maltreatment with WHtR in sex-, region- and weight-stratified analyses. Future studies in clinical populations are warranted to examine U-shaped correlations between increased WHtR and childhood maltreatment.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Adverse Childhood Experiences", "major": "N"}, {"term": "Child", "major": "N"}, {"term": "Child Abuse", "major": "N"}, {"term": "Disease Susceptibility", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Multivariate Analysis", "major": "N"}, {"term": "Obesity", "major": "N"}, {"term": "Population Surveillance", "major": "N"}, {"term": "Risk Assessment", "major": "N"}, {"term": "Risk Factors", "major": "N"}], "other": []}, "full_author_list": ["Fleischer T", "Ulke C", "Beutel M", "Binder H", "Br\u00e4hler E", "Johar H", "Atasoy S", "Kruse J", "Otten D", "Tibubos A", "Z\u00f6ller D", "Speerforck S", "Grabe H", "Ladwig K", "Schomerus G"], "journal": {"journal_name": "Sci Rep", "volume": "11", "issue": "1"}}, {"IDs": {"DOI": "10.1093/aje/kwad206", "PMID": "37856700"}, "title": "Associations of Maternal Educational Level, Proximity to Green Space During Pregnancy, and Gestational Diabetes With Body Mass Index From Infancy to Early Adulthood: A Proof-of-Concept Federated Analysis in 18 Birth Cohorts.", "abstract": "['International sharing of cohort data for research is important and challenging. We explored the feasibility of multicohort federated analyses by examining associations between 3 pregnancy exposures (maternal education, exposure to green vegetation, and gestational diabetes) and offspring body mass index (BMI) from infancy to age 17 years. We used data from 18 cohorts (n\\xa0= 206,180 mother-child pairs) from the EU Child Cohort Network and derived BMI at ages 0-1, 2-3, 4-7, 8-13, and 14-17 years. Associations were estimated using linear regression via 1-stage individual participant data meta-analysis using DataSHIELD. Associations between lower maternal education and higher child BMI emerged from age 4 and increased with age (difference in BMI z score comparing low with high education, at age 2-3 years\\xa0= 0.03 (95% confidence interval (CI): 0.00, 0.05), at 4-7 years\\xa0= 0.16 (95% CI: 0.14, 0.17), and at 8-13 years\\xa0= 0.24 (95% CI: 0.22, 0.26)). Gestational diabetes was positively associated with BMI from age 8 years (BMI z score difference\\xa0= 0.18, 95% CI: 0.12, 0.25) but not at younger ages; however, associations attenuated towards the null when restricted to cohorts that measured gestational diabetes via universal screening. Exposure to green vegetation was weakly associated with higher BMI up to age 1 year but not at older ages. Opportunities of cross-cohort federated analyses are discussed.']", "keywords": {"mesh": [{"term": "Humans", "major": "N"}, {"term": "Diabetes, Gestational", "major": "Y"}, {"term": "Female", "major": "N"}, {"term": "Pregnancy", "major": "N"}, {"term": "Body Mass Index", "major": "Y"}, {"term": "Adolescent", "major": "N"}, {"term": "Infant", "major": "N"}, {"term": "Child, Preschool", "major": "N"}, {"term": "Child", "major": "N"}, {"term": "Educational Status", "major": "Y"}, {"term": "Male", "major": "N"}, {"term": "Prenatal Exposure Delayed Effects", "major": "N"}, {"term": "Birth Cohort", "major": "N"}, {"term": "Infant, Newborn", "major": "N"}, {"term": "Adult", "major": "N"}, {"term": "Proof of Concept Study", "major": "N"}, {"term": "Parks, Recreational", "major": "N"}], "other": ["BMI", "cohort data sharing", "federated analyses", "gestational diabetes", "green spaces", "maternal education"]}, "full_author_list": ["Cadman T", "Elhakeem A", "Vinther J", "Avraam D", "Carrasco P", "Calas L", "Cardol M", "Charles M", "Corpeleijn E", "Crozier S", "de Castro M", "Estarlich M", "Fernandes A", "Fossatti S", "Gruszfeld D", "Guerlich K", "Grote V", "Haakma S", "Harris J", "Heude B", "Huang R", "Ibarluzea J", "Inskip H", "Jaddoe V", "Koletzko B", "Lioret S", "Luque V", "Manios Y", "Moirano G", "Moschonis G", "Nader J", "Nieuwenhuijsen M", "Andersen A", "McEachen R", "de Moira A", "Popovic M", "Roumeliotaki T", "Salika T", "Santa Marina L", "Santos S", "Serbert S", "Tzorovili E", "Vafeiadi M", "Verduci E", "Vrijheid M", "Vrijkotte T", "Welten M", "Wright J", "Yang T", "Zugna D", "Lawlor D"], "journal": {"journal_name": "Am J Epidemiol", "volume": "193", "issue": "5"}}, {"IDs": {"DOI": "10.1159/000368959", "PMID": "25532061"}, "title": "DataSHIELD: an ethically robust solution to multiple-site individual-level data analysis.", "abstract": "['DataSHIELD (Data Aggregation Through Anonymous Summary-statistics from Harmonised Individual levEL Databases) has been proposed to facilitate the co-analysis of individual-level data from multiple studies without physically sharing the data. In a previous paper, we investigated whether DataSHIELD could protect participant confidentiality in accordance with UK law. In this follow-up paper, we investigate whether DataSHIELD addresses a broader range of ethics-related data-sharing concerns.', 'Ethics-related data-sharing concerns of Institutional Review Boards, ethics experts, international research consortia and research participants were identified through a literature search and systematically examined at a multidisciplinary workshop to determine whether DataSHIELD proposes mechanisms which can address these concerns.', \"DataSHIELD addresses several ethics-related data-sharing concerns related to privacy, confidentiality, and the protection of the research participant's rights while sharing data and after the data have been shared. The data remain entirely under the direct management of the study that collected them. Data processing commands are strictly supervised, and the data are queried in a protected environment. Issues related to the return of individual research results when data are shared are eliminated; the responsibility for return remains at the study of origin.\", 'DataSHIELD can provide an innovative and robust solution for addressing commonly encountered ethics-related data-sharing concerns.']", "keywords": {"mesh": [{"term": "Access to Information", "major": "N"}, {"term": "Biomedical Research", "major": "N"}, {"term": "Confidentiality", "major": "N"}, {"term": "Data Collection", "major": "Y"}, {"term": "Databases, Factual", "major": "N"}, {"term": "Ethics Committees, Research", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Dissemination", "major": "Y"}], "other": []}, "full_author_list": ["Budin-Lj\u00f8sne I", "Burton P", "Isaeva J", "Gaye A", "Turner A", "Murtagh M", "Wallace S", "Ferretti V", "Harris J"], "journal": {"journal_name": "Public Health Genomics", "volume": "18", "issue": "2"}}, {"IDs": {"DOI": "10.1371/journal.pone.0223010", "PMID": "31581246"}, "title": "KETOS: Clinical decision support and machine learning as a service - A training and deployment platform based on Docker, OMOP-CDM, and FHIR Web Services.", "abstract": "['To take full advantage of decision support, machine learning, and patient-level prediction models, it is important that models are not only created, but also deployed in a clinical setting. The KETOS platform demonstrated in this work implements a tool for researchers allowing them to perform statistical analyses and deploy resulting models in a secure environment.', 'The proposed system uses Docker virtualization to provide researchers with reproducible data analysis and development environments, accessible via Jupyter Notebook, to perform statistical analysis and develop, train and deploy models based on standardized input data. The platform is built in a modular fashion and interfaces with web services using the Health Level 7 (HL7) Fast Healthcare Interoperability Resources (FHIR) standard to access patient data. In our prototypical implementation we use an OMOP common data model (OMOP-CDM) database. The architecture supports the entire research lifecycle from creating a data analysis environment, retrieving data, and training to final deployment in a hospital setting.', 'We evaluated the platform by establishing and deploying an analysis and end user application for hemoglobin reference intervals within the University Hospital Erlangen. To demonstrate the potential of the system to deploy arbitrary models, we loaded a colorectal cancer dataset into an OMOP database and built machine learning models to predict patient outcomes and made them available via a web service. We demonstrated both the integration with FHIR as well as an example end user application. Finally, we integrated the platform with the open source DataSHIELD architecture to allow for distributed privacy preserving data analysis and training across networks of hospitals.', 'The KETOS platform takes a novel approach to data analysis, training and deploying decision support models in a hospital or healthcare setting. It does so in a secure and privacy-preserving manner, combining the flexibility of Docker virtualization with the advantages of standardized vocabularies, a widely applied database schema (OMOP-CDM), and a standardized way to exchange medical data (FHIR).']", "keywords": {"mesh": [{"term": "Colorectal Neoplasms", "major": "N"}, {"term": "Decision Support Systems, Clinical", "major": "Y"}, {"term": "Health Information Interoperability", "major": "Y"}, {"term": "Hemoglobins", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Internet", "major": "Y"}, {"term": "Machine Learning", "major": "Y"}, {"term": "Models, Theoretical", "major": "Y"}, {"term": "Privacy", "major": "N"}, {"term": "Reference Values", "major": "N"}, {"term": "Treatment Outcome", "major": "N"}, {"term": "User-Computer Interface", "major": "N"}], "other": []}, "full_author_list": ["Gruendner J", "Schwachhofer T", "Sippl P", "Wolf N", "Erpenbeck M", "Gulden C", "Kapsner L", "Zierk J", "Mate S", "St\u00fcrzl M", "Croner R", "Prokosch H", "Toddenroth D"], "journal": {"journal_name": "PLoS One", "volume": "14", "issue": "10"}}, {"IDs": {"DOI": "10.1002/sta4.19", "PMID": ""}, "title": "Combined analysis of correlated data when data cannot be pooled", "abstract": "In genetic epidemiology studies, associations between individual genetic variants and phenotypes of interest are generally weak requiring large samples to estimate effects and to address complex statistical questions. Such sample sizes are often only achievable by pooling data from multiple studies; effects of interest can then be investigated through an individual\u2010level meta\u2010analysis (ILMA) on the pooled data, or by conducting a conventional study\u2010level meta\u2010analysis (SLMA). However, pooling individual\u2010level research data for an ILMA is not always possible, and researchers may be compelled to conduct an SLMA instead, restricting the sharing to non\u2010disclosing summary statistics. In certain settings, an individual\u2010level analysis can be conducted without pooling the data from the different studies. It has already been shown that when data are horizontally partitioned between studies, i.e. data are collected on the same variables in each study but any given study participant appears in one study only, it is possible to fit a generalised linear model in this way. In the present paper, we demonstrate that an individual\u2010level generalised estimating equations meta\u2010analysis can be achieved in an analogous manner. This extends the scope of ILMA without data pooling to problems involving correlated and clustered responses. Copyright \u00a9 2013 John Wiley & Sons Ltd", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Jones E", "Sheehan N", "Gaye A", "Laflamme P", "Burton P"], "journal": {"journal_name": "Stat", "volume": "2", "issue": "1"}}, {"IDs": {"DOI": "10.1038/ejhg.2014.196", "PMID": "25248396"}, "title": "Genomic cloud computing: legal and ethical points to consider.", "abstract": "[\"The biggest challenge in twenty-first century data-intensive genomic science, is developing vast computer infrastructure and advanced software tools to perform comprehensive analyses of genomic data sets for biomedical research and clinical practice. Researchers are increasingly turning to cloud computing both as a solution to integrate data from genomics, systems biology and biomedical data mining and as an approach to analyze data to solve biomedical problems. Although cloud computing provides several benefits such as lower costs and greater efficiency, it also raises legal and ethical issues. In this article, we discuss three key 'points to consider' (data control; data security, confidentiality and transfer; and accountability) based on a preliminary review of several publicly available cloud service providers' Terms of Service. These 'points to consider' should be borne in mind by genomic research organizations when negotiating legal arrangements to store genomic data on a large commercial cloud service provider's servers. Diligent genomic cloud computing means leveraging security standards and evaluation processes as a means to protect data and entails many of the same good practices that researchers should always consider in securing their local infrastructure.\"]", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "N"}, {"term": "Cloud Computing", "major": "Y"}, {"term": "Computer Security", "major": "N"}, {"term": "Confidentiality", "major": "N"}, {"term": "Data Mining", "major": "N"}, {"term": "Genomics", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Internet", "major": "N"}, {"term": "Software", "major": "N"}], "other": []}, "full_author_list": ["Dove E", "Joly Y", "Tass\u00e9 A", "Knoppers B"], "journal": {"journal_name": "Eur J Hum Genet", "volume": "23", "issue": "10"}}, {"IDs": {"DOI": "10.1136/bmjopen-2021-055630", "PMID": "34794999"}, "title": "Unravelling data for rapid evidence-based response to COVID-19: a summary of the unCoVer protocol.", "abstract": "['unCoVer-Unravelling data for rapid evidence-based response to COVID-19-is a Horizon 2020-funded network of 29 partners from 18 countries capable of collecting and using real-world data (RWD) derived from the response and provision of care to patients with COVID-19 by health systems across Europe and elsewhere. unCoVer aims to exploit the full potential of this information to rapidly address clinical and epidemiological research questions arising from the evolving pandemic.', \"From the onset of the COVID-19 pandemic, partners are gathering RWD from electronic health records currently including information from over 22 000 hospitalised patients with COVID-19, and national surveillance and screening data, and registries with over 1 900 000 COVID-19 cases across Europe, with continuous updates. These heterogeneous datasets will be described, harmonised and integrated into a multi-user data repository operated through Opal-DataSHIELD, an interoperable open-source server application. Federated data analyses, without sharing or disclosing any individual-level data, will be performed with the objective to reveal patients' baseline characteristics, biomarkers, determinants of COVID-19 prognosis, safety and effectiveness of treatments, and potential strategies against COVID-19, as well as epidemiological patterns. These analyses will complement evidence from efficacy/safety clinical trials, where vulnerable, more complex/heterogeneous populations and those most at risk of severe COVID-19 are often excluded.\", \"After strict ethical considerations, databases will be available through a federated data analysis platform that allows processing of available COVID-19 RWD without disclosing identification information to analysts and limiting output to data aggregates. Dissemination of unCoVer's activities will be related to the access and use of dissimilar RWD, as well as the results generated by the pooled analyses. Dissemination will include training and educational activities, scientific publications and conference communications.\"]", "keywords": {"mesh": [{"term": "COVID-19", "major": "Y"}, {"term": "Europe", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Pandemics", "major": "Y"}, {"term": "SARS-CoV-2", "major": "N"}], "other": ["COVID-19", "public health", "statistics & research methods"]}, "full_author_list": ["Pe\u00f1alvo J", "Mertens E", "Ademovi\u0107 E", "Akgun S", "Baltazar A", "Buonfrate D", "\u010coklo M", "Devleesschauwer B", "Diaz Valencia P", "Fernandes J", "G\u00f3mez E", "Hynds P", "Kabir Z", "Klein J", "Kostoulas P", "Llanos Jim\u00e9nez L", "Lotrean L", "Majdan M", "Menasalvas E", "Nguewa P", "Oh I", "O'Sullivan G", "Pereira D", "Reina Ortiz M", "Riva S", "Soriano G", "Soriano J", "Spilki F", "Tamang M", "Trofor A", "Vaillant M", "Van Ierssel S", "Vukovi\u0107 J", "Castellano J"], "journal": {"journal_name": "BMJ Open", "volume": "11", "issue": "11"}}, {"IDs": {"DOI": "10.1007/s00787-023-02149-0", "PMID": "36749392"}, "title": "Sleep duration in preschool age and later behavioral and cognitive outcomes: an individual participant data meta-analysis in five European cohorts.", "abstract": "[\"Short sleep duration has been linked to adverse behavioral and cognitive outcomes in schoolchildren, but few studies examined this relation in preschoolers. We aimed to investigate the association between parent-reported sleep duration at 3.5\\xa0years and behavioral and cognitive outcomes at 5\\xa0years in European children. We used harmonized data from five cohorts of the European Union Child Cohort Network: ALSPAC, SWS (UK); EDEN, ELFE (France); INMA (Spain). Associations were estimated through DataSHIELD using adjusted generalized linear regression models fitted separately for each cohort and pooled with random-effects meta-analysis. Behavior was measured with the Strengths and Difficulties Questionnaire. Language and non-verbal intelligence were assessed by the Wechsler Preschool and Primary Scale of Intelligence or the McCarthy Scales of Children's Abilities. Behavioral and cognitive analyses included 11,920 and 2981 children, respectively (34.0%/13.4% of the original sample). In meta-analysis, longer mean sleep duration per day at 3.5\\xa0years was associated with lower mean internalizing and externalizing behavior percentile scores at 5\\xa0years (adjusted mean difference: -\\xa01.27, 95% CI [-\\xa02.22, -\\xa00.32] / -\\xa02.39, 95% CI [-\\xa03.04, -\\xa01.75]). Sleep duration and language or non-verbal intelligence showed trends of inverse associations, however, with imprecise estimates (adjusted mean difference: -\\xa00.28, 95% CI [-\\xa00.83, 0.27] / -\\xa00.42, 95% CI [-\\xa00.99, 0.15]). This individual participant data meta-analysis suggests that longer sleep duration in preschool age may be important for children's later behavior and highlight the need for larger samples for robust analyses of cognitive outcomes. Findings could be influenced by confounding or reverse causality and require replication.\"]", "keywords": {"mesh": [{"term": "Child", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Child, Preschool", "major": "N"}, {"term": "Sleep Duration", "major": "Y"}, {"term": "Language", "major": "Y"}, {"term": "Wechsler Scales", "major": "N"}, {"term": "Sleep", "major": "N"}, {"term": "Cognition", "major": "N"}], "other": ["Externalizing behavior", "Internalizing behavior", "Language", "Multi-cohort analysis", "Non-verbal intelligence", "Preschool sleep duration"]}, "full_author_list": ["Guerlich K", "Avraam D", "Cadman T", "Calas L", "Charles M", "Elhakeem A", "Fern\u00e1ndez-Barr\u00e9s S", "Guxens M", "Heude B", "Ibarluzea J", "Inskip H", "Julvez J", "Lawlor D", "Murcia M", "Salika T", "Sunyer J", "Tafflet M", "Koletzko B", "Grote V", "Plancoulaine S"], "journal": {"journal_name": "Eur Child Adolesc Psychiatry", "volume": "33", "issue": "1"}}, {"IDs": {"DOI": "10.1186/s13104-022-06085-1", "PMID": "35659747"}, "title": "dsSurvival: Privacy preserving survival models for federated individual patient meta-analysis in DataSHIELD.", "abstract": "['Achieving sufficient statistical power in a survival analysis usually requires large amounts of data from different sites. Sensitivity of individual-level data, ethical and practical considerations regarding data sharing across institutions could be a potential challenge for achieving this added power. Hence we implemented a federated meta-analysis approach of survival models in DataSHIELD, where only anonymous aggregated data are shared across institutions, while simultaneously allowing for exploratory, interactive modelling. In this case, meta-analysis techniques to combine analysis results from each site are a solution, but an analytic workflow involving local analysis undertaken at individual studies hinders exploration. Thus, the aim is to provide a framework for performing meta-analysis of Cox regression models across institutions without manual analysis steps for the data providers.', 'We introduce a package (dsSurvival) which allows privacy preserving meta-analysis of survival models, including the calculation of hazard ratios. Our tool can be of great use in biomedical research where there is a need for building survival models and there are privacy concerns about sharing data.']", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "Y"}, {"term": "Humans", "major": "N"}, {"term": "Information Dissemination", "major": "N"}, {"term": "Privacy", "major": "Y"}], "other": ["Federated analysis", "Meta-analysis", "Survival analysis"]}, "full_author_list": ["Banerjee S", "Sofack G", "Papakonstantinou T", "Avraam D", "Burton P", "Z\u00f6ller D", "Bishop T"], "journal": {"journal_name": "BMC Res Notes", "volume": "15", "issue": "1"}}, {"IDs": {"DOI": "10.1007/s00439-011-1036-3", "PMID": "21706184"}, "title": "Realizing the promise of population biobanks: a new model for translation.", "abstract": "[\"The promise of science lies in expectations of its benefits to societies and is matched by expectations of the realisation of the significant public investment in that science. In this paper, we undertake a methodological analysis of the science of biobanking and a sociological analysis of translational research in relation to biobanking. Part of global and local endeavours to translate raw biomedical evidence into practice, biobanks aim to provide a platform for generating new scientific knowledge to inform development of new policies, systems and interventions to enhance the public's health. Effectively translating scientific knowledge into routine practice, however, involves more than good science. Although biobanks undoubtedly provide a fundamental resource for both clinical and public health practice, their potentiating ontology--that their outputs are perpetually a promise of scientific knowledge generation--renders translation rather less straightforward than drug discovery and treatment implementation. Biobanking science, therefore, provides a perfect counterpoint against which to test the bounds of translational research. We argue that translational research is a contextual and cumulative process: one that is necessarily dynamic and interactive and involves multiple actors. We propose a new multidimensional model of translational research which enables us to imagine a new paradigm: one that takes us from bench to bedside to backyard and beyond, that is, attentive to the social and political context of translational science, and is cognisant of all the players in that process be they researchers, health professionals, policy makers, industry representatives, members of the public or research participants, amongst others.\"]", "keywords": {"mesh": [{"term": "Biological Specimen Banks", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Models, Biological", "major": "N"}, {"term": "Public Health", "major": "N"}, {"term": "Public Policy", "major": "N"}, {"term": "Translational Research, Biomedical", "major": "Y"}], "other": []}, "full_author_list": ["Murtagh M", "Demir I", "Harris J", "Burton P"], "journal": {"journal_name": "Hum Genet", "volume": "130", "issue": "3"}}, {"IDs": {"DOI": "10.1140/epjds/s13688-020-00257-4", "PMID": "33442528"}, "title": "Privacy preserving data visualizations.", "abstract": "['Data visualizations are a valuable tool used during both statistical analysis and the interpretation of results as they graphically reveal useful information about the structure, properties and relationships between variables, which may otherwise be concealed in tabulated data. In disciplines like medicine and the social sciences, where collected data include sensitive information about study participants, the sharing and publication of individual-level records is controlled by data protection laws and ethico-legal norms. Thus, as data visualizations - such as graphs and plots - may be linked to other released information and used to identify study participants and their personal attributes, their creation is often prohibited by the terms of data use. These restrictions are enforced to reduce the risk of breaching data subject confidentiality, however they limit analysts from displaying useful descriptive plots for their research features and findings. Here we propose the use of anonymization techniques to generate privacy-preserving visualizations that retain the statistical properties of the underlying data while still adhering to strict data disclosure rules. We demonstrate the use of (i)\\xa0the well-known &lt;i&gt;k&lt;/i&gt;-anonymization process which preserves privacy by reducing the granularity of the data using suppression and generalization, (ii) a novel deterministic approach that replaces individual-level observations with the centroids of each &lt;i&gt;k&lt;/i&gt; nearest neighbours, and (iii) a probabilistic procedure that perturbs individual attributes with the addition of random stochastic noise. We apply the proposed methods to generate privacy-preserving data visualizations for exploratory data analysis and inferential regression plot diagnostics, and we discuss their strengths and limitations.']", "keywords": {"mesh": [], "other": ["Anonymization", "Data visualizations", "Disclosure control", "Privacy protection", "Sensitive data"]}, "full_author_list": ["Avraam D", "Wilson R", "Butters O", "Burton T", "Nicolaides C", "Jones E", "Boyd A", "Burton P"], "journal": {"journal_name": "EPJ Data Sci", "volume": "10", "issue": "1"}}, {"IDs": {"DOI": "10.1016/j.envint.2024.108684", "PMID": ""}, "title": "Green space exposure and blood DNA methylation at birth and in childhood \u2013 A multi-cohort study", "abstract": "", "keywords": {"mesh": [], "other": []}, "full_author_list": ["Aguilar-Lacasa\u00f1a S", "Fontes Marques I", "de Castro M", "Dadvand P", "Escrib\u00e0 X", "Fossati S", "Gonz\u00e1lez J", "Nieuwenhuijsen M", "Alfano R", "Annesi-Maesano I", "Brescianini S", "Burrows K", "Calas L", "Elhakeem A", "Heude B", "Hough A", "Isaevska E", "W V Jaddoe V", "Lawlor D", "Monaghan G", "Nawrot T", "Plusquin M", "Richiardi L", "Watmuff A", "Yang T", "Vrijheid M", "F Felix J", "Bustamante M"], "journal": {"journal_name": "Environment International", "volume": "188", "issue": ""}}, {"IDs": {"DOI": "10.1183/13993003.02127-2015", "PMID": "27824608"}, "title": "Ambient air pollution, traffic noise and adult asthma prevalence: a BioSHaRE approach.", "abstract": "['We investigated the effects of both ambient air pollution and traffic noise on adult asthma prevalence, using harmonised data from three European cohort studies established in 2006-2013 (HUNT3, Lifelines and UK Biobank).Residential exposures to ambient air pollution (particulate matter with aerodynamic diameter \u226410\\u2005\u00b5m (PM&lt;sub&gt;10&lt;/sub&gt;) and nitrogen dioxide (NO&lt;sub&gt;2&lt;/sub&gt;)) were estimated by a pan-European Land Use Regression model for 2007. Traffic noise for 2009 was modelled at home addresses by adapting a standardised noise assessment framework (CNOSSOS-EU). A cross-sectional analysis of 646\\u200a731 participants aged \u226520\\u2005years was undertaken using DataSHIELD to pool data for individual-level analysis via a \"compute to the data\" approach. Multivariate logistic regression models were fitted to assess the effects of each exposure on lifetime and current asthma prevalence.PM&lt;sub&gt;10&lt;/sub&gt; or NO&lt;sub&gt;2&lt;/sub&gt; higher by 10\\u2005\u00b5g\u00b7m&lt;sup&gt;-3&lt;/sup&gt; was associated with 12.8% (95% CI 9.5-16.3%) and 1.9% (95% CI 1.1-2.8%) higher lifetime asthma prevalence, respectively, independent of confounders. Effects were larger in those aged \u226550\\u2005years, ever-smokers and less educated. Noise exposure was not significantly associated with asthma prevalence.This study suggests that long-term ambient PM&lt;sub&gt;10&lt;/sub&gt; exposure is associated with asthma prevalence in western European adults. Traffic noise is not associated with asthma prevalence, but its potential to impact on asthma exacerbations needs further investigation.']", "keywords": {"mesh": [{"term": "Adult", "major": "N"}, {"term": "Aged", "major": "N"}, {"term": "Aged, 80 and over", "major": "N"}, {"term": "Air Pollutants", "major": "N"}, {"term": "Air Pollution", "major": "N"}, {"term": "Asthma", "major": "N"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Cross-Sectional Studies", "major": "N"}, {"term": "Environmental Exposure", "major": "N"}, {"term": "Environmental Monitoring", "major": "N"}, {"term": "European Union", "major": "N"}, {"term": "Female", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Logistic Models", "major": "N"}, {"term": "Male", "major": "N"}, {"term": "Middle Aged", "major": "N"}, {"term": "Multivariate Analysis", "major": "N"}, {"term": "Nitrogen Dioxide", "major": "N"}, {"term": "Noise", "major": "N"}, {"term": "Particulate Matter", "major": "N"}, {"term": "Transportation", "major": "Y"}, {"term": "Young Adult", "major": "N"}], "other": []}, "full_author_list": ["Cai Y", "Zijlema W", "Doiron D", "Blangiardo M", "Burton P", "Fortier I", "Gaye A", "Gulliver J", "de Hoogh K", "Hveem K", "Mbatchou S", "Morley D", "Stolk R", "Elliott P", "Hansell A", "Hodgson S"], "journal": {"journal_name": "Eur Respir J", "volume": "49", "issue": "1"}}, {"IDs": {"DOI": "10.1159/000336673", "PMID": "22722688"}, "title": "Securing the data economy: translating privacy and enacting security in the development of DataSHIELD.", "abstract": "[\"Contemporary bioscience is seeing the emergence of a new data economy: with data as its fundamental unit of exchange. While sharing data within this new 'economy' provides many potential advantages, the sharing of individual data raises important social and ethical concerns. We examine ongoing development of one technology, DataSHIELD, which appears to elide privacy concerns about sharing data by enabling shared analysis while not actually sharing any individual-level data. We combine presentation of the development of DataSHIELD with presentation of an ethnographic study of a workshop to test the technology. DataSHIELD produced an application of the norm of privacy that was practical, flexible and operationalizable in researchers' everyday activities, and one which fulfilled the requirements of ethics committees. We demonstrated that an analysis run via DataSHIELD could precisely replicate results produced by a standard analysis where all data are physically pooled and analyzed together. In developing DataSHIELD, the ethical concept of privacy was transformed into an issue of security. Development of DataSHIELD was based on social practices as well as scientific and ethical motivations. Therefore, the 'success' of DataSHIELD would, likewise, be dependent on more than just the mathematics and the security of the technology.\"]", "keywords": {"mesh": [{"term": "Biomedical Research", "major": "Y"}, {"term": "Computer Security", "major": "N"}, {"term": "Confidentiality", "major": "N"}, {"term": "Ethics Committees", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Storage and Retrieval", "major": "N"}, {"term": "Research", "major": "N"}, {"term": "Research Design", "major": "Y"}], "other": []}, "full_author_list": ["Murtagh M", "Demir I", "Jenkings K", "Wallace S", "Murtagh B", "Boniol M", "Bota M", "Laflamme P", "Boffetta P", "Ferretti V", "Burton P"], "journal": {"journal_name": "Public Health Genomics", "volume": "15", "issue": "5"}}, {"IDs": {"DOI": "10.1186/s12889-018-5031-7", "PMID": "29351781"}, "title": "MINDMAP: establishing an integrated database infrastructure for research in ageing, mental well-being, and the urban environment.", "abstract": "['Urbanization and ageing have important implications for public mental health and well-being. Cities pose major challenges for older citizens, but also offer opportunities to develop, test, and implement policies, services, infrastructure, and interventions that promote mental well-being. The MINDMAP project aims to identify the opportunities and challenges posed by urban environmental characteristics for the promotion and management of mental well-being and cognitive function of older individuals.', 'MINDMAP aims to achieve its research objectives by bringing together longitudinal studies from 11 countries covering over 35 cities linked to databases of area-level environmental exposures and social and urban policy indicators. The infrastructure supporting integration of this data will allow multiple MINDMAP investigators to safely and remotely co-analyse individual-level and area-level data. Individual-level data is derived from baseline and follow-up measurements of ten participating cohort studies and provides information on mental well-being outcomes, sociodemographic variables, health behaviour characteristics, social factors, measures of frailty, physical function indicators, and chronic conditions, as well as blood derived clinical biochemistry-based biomarkers and genetic biomarkers. Area-level information on physical environment characteristics (e.g. green spaces, transportation), socioeconomic and sociodemographic characteristics (e.g. neighbourhood income, residential segregation, residential density), and social environment characteristics (e.g. social cohesion, criminality) and national and urban social policies is derived from publically available sources such as geoportals and administrative databases. The linkage, harmonization, and analysis of data from different sources are being carried out using piloted tools to optimize the validity of the research results and transparency of the methodology.', 'MINDMAP is a novel research collaboration that is combining population-based cohort data with publicly available datasets not typically used for ageing and mental well-being research. Integration of various data sources and observational units into a single platform will help to explain the differences in ageing-related mental and cognitive disorders both within as well as between cities in Europe, the US, Canada, and Russia and to assess the causal pathways and interactions between the urban environment and the individual determinants of mental well-being and cognitive ageing in older adults.']", "keywords": {"mesh": [{"term": "Aging", "major": "Y"}, {"term": "Canada", "major": "N"}, {"term": "Cities", "major": "Y"}, {"term": "Cohort Studies", "major": "N"}, {"term": "Databases as Topic", "major": "N"}, {"term": "Europe", "major": "N"}, {"term": "Humans", "major": "N"}, {"term": "Information Storage and Retrieval", "major": "N"}, {"term": "Mental Health", "major": "Y"}, {"term": "Research", "major": "N"}, {"term": "Russia", "major": "N"}, {"term": "United States", "major": "N"}, {"term": "Urban Health", "major": "N"}], "other": ["Ageing", "Cohort studies", "Data integration", "Database", "Mental well-being", "Urban health"]}, "full_author_list": ["Beenackers M", "Doiron D", "Fortier I", "Noordzij J", "Reinhard E", "Courtin E", "Bobak M", "Chaix B", "Costa G", "Dapp U", "Diez Roux A", "Huisman M", "Grundy E", "Krokstad S", "Martikainen P", "Raina P", "Avendano M", "van Lenthe F"], "journal": {"journal_name": "BMC Public Health", "volume": "18", "issue": "1"}}, {"IDs": {"DOI": "10.1038/ejhg.2012.96", "PMID": "22713808"}, "title": "Toward a roadmap in global biobanking for health.", "abstract": "['Biobanks can have a pivotal role in elucidating disease etiology, translation, and advancing public health. However, meeting these challenges hinges on a critical shift in the way science is conducted and requires biobank harmonization. There is growing recognition that a common strategy is imperative to develop biobanking globally and effectively. To help guide this strategy, we articulate key principles, goals, and priorities underpinning a roadmap for global biobanking to accelerate health science, patient care, and public health. The need to manage and share very large amounts of data has driven innovations on many fronts. Although technological solutions are allowing biobanks to reach new levels of integration, increasingly powerful data-collection tools, analytical techniques, and the results they generate raise new ethical and legal issues and challenges, necessitating a reconsideration of previous policies, practices, and ethical norms. These manifold advances and the investments that support them are also fueling opportunities for biobanks to ultimately become integral parts of health-care systems in many countries. International harmonization to increase interoperability and sustainability are two strategic priorities for biobanking. Tackling these issues requires an environment favorably inclined toward scientific funding and equipped to address socio-ethical challenges. Cooperation and collaboration must extend beyond systems to enable the exchange of data and samples to strategic alliances between many organizations, including governmental bodies, funding agencies, public and private science enterprises, and other stakeholders, including patients. A common vision is required and we articulate the essential basis of such a vision herein.']", "keywords": {"mesh": [{"term": "Biological Specimen Banks", "major": "N"}, {"term": "Data Collection", "major": "N"}, {"term": "Databases, Factual", "major": "N"}], "other": []}, "full_author_list": ["Harris J", "Burton P", "Knoppers B", "Lindpaintner K", "Bledsoe M", "Brookes A", "Budin-Lj\u00f8sne I", "Chisholm R", "Cox D", "Desch\u00eanes M", "Fortier I", "Hainaut P", "Hewitt R", "Kaye J", "Litton J", "Metspalu A", "Ollier B", "Palmer L", "Palotie A", "Pasterk M", "Perola M", "Riegman P", "van Ommen G", "Yuille M", "Zatloukal K"], "journal": {"journal_name": "Eur J Hum Genet", "volume": "20", "issue": "11"}}];
                </script>
            </div>
        </div>
        <div class="foot">
            <span id="update_timestamp"><a href="..//timestamp.txt">Update time</a></span>
            <script type="text/javascript" src="..//timestamp.js"></script>
            <script>
            update_timestamp("..//timestamp.txt")
            </script>
        </div>
    </body>
</html>
